<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <title>Statistical analysis of DIA data</title>
        
            <script async defer data-domain="training.galaxyproject.org" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>

        
        <link rel="stylesheet" href="/training-material/assets/css/bootstrap.min.css?v=3">
        <link rel="stylesheet" href="/training-material/assets/css/bootstrap-toc.min.css">
        <link rel="stylesheet" href="/training-material/assets/css/main.css?v=2">
        <script src="https://kit.fontawesome.com/67b3f98409.js" crossorigin="anonymous"></script>
        <link rel="stylesheet" href="/training-material/assets/css/academicons.css">
        <link rel="stylesheet" href="/training-material/assets/css/syntax_highlighting.css">
        <link rel="shortcut icon" href="/training-material/favicon.ico" type="image/x-icon" />
        <link rel="alternate" type="application/atom+xml" href="/training-material/feed.xml" />

        
        
        
        
        
        <meta name="description" content="Training material for proteomics workflows in Galaxy" />
        <meta property="og:title" content="Galaxy Training: Statistical analysis of DIA data" />
        <meta property="og:description" content="Training material for proteomics workflows in Galaxy" />
        <meta property="og:image" content="/training-material/assets/images/GTNLogo1000.png" />
    </head>
    <body data-spy="scroll" data-target="#toc">
        <header>
    <nav class="navbar navbar-expand-md navbar-dark" aria-label="Site Navigation">
        <div class="container">
            <a class="navbar-brand" href="/training-material/">
                <img src="/training-material/assets/images/GTN-60px.png" height="30" alt="Galaxy Training Network logo">
                Galaxy Training!
            </a>

            <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#top-navbar" aria-controls="top-navbar" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="top-navbar">
                <ul class="navbar-nav">
                    <li class="nav-item">
                        
                        <a class="nav-link" href="/training-material/topics/proteomics" title="Go back to list of tutorials">
                            <i class="far fa-folder" aria-hidden="true"></i> Proteomics
                        </a>
                        
                    </li>

                    <li class="nav-item dropdown">
                        <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Languages">
                            <i class="fas fa-language" aria-hidden="true"></i><span class="visually-hidden">language</span> Languages
                        </a>
                        <div class="dropdown-menu dropdown-menu-right">
                            
                              <strong class="dropdown-header">Automatic Translations</strong>
            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=fr&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url">
                            Fran√ßais
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=ja&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url">
                            Êó•Êú¨Ë™û
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=es&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url">
                            Espa√±ol
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=pt&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url">
                            Portugu√™s
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=ar&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url">
                            ÿßŸÑÿπÿ±ÿ®Ÿäÿ©
                            </a>
                            
                            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html&edit-text=&act=url" title="">
                            And more!
                            </a>
                        </div>
                    </li>

                    <li class="nav-item dropdown">
    <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Help">
        <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">help</span> Help
    </a>
    <div class="dropdown-menu dropdown-menu-right">
        <!-- disable Tess for now
        <form method="get" action="https://tess.elixir-europe.org/materials">
            <input type="text" id="search" name="q" value="" style="margin-left: 0.5em;/*! border-radius: 0px; */">
            <input type="hidden" value="Galaxy Training" name="content_provider">
            <input type="submit" value="Search on TeSS" style="width: 92%;border-radius: 0px;margin: 0.5em;background: #f47d20;border: 0px;padding: 0.25em;" class="">
        </form>
        -->

        <a class="dropdown-item" href="/training-material/faq" title="Check our FAQs">
           <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> FAQs
        </a>
        
        <a class="dropdown-item" href="/training-material/topics/proteomics/faqs/" title="Check our FAQs for the Proteomics topic">
           <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> Topic FAQs
        </a>
        
        <a class="dropdown-item" href="https://help.galaxyproject.org/" title="Discuss on Galaxy Help">
            <i class="far fa-comments" aria-hidden="true"></i><span class="visually-hidden">feedback</span> Galaxy Help Forum
        </a>
        <a class="dropdown-item" href="https://gitter.im/Galaxy-Training-Network/Lobby" title="Discuss on gitter">
           <i class="fab fa-gitter" aria-hidden="true"></i><span class="visually-hidden">gitter</span> Discuss on Gitter
        </a>
    </div>
</li>


                    <li class="nav-item dropdown">
    <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Extras">
        <i class="far fa-star" aria-hidden="true"></i><span class="visually-hidden">galaxy-star</span> Extras
    </a>
    <div class="dropdown-menu dropdown-menu-right">

        
        <a class="dropdown-item" href="https://github.com/galaxyproject/training-material/edit/main/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.md" title="Edit on GitHub">
          <i class="fab fa-github" aria-hidden="true"></i><span class="visually-hidden">github</span> Edit on GitHub
        </a>

        <a class="dropdown-item" href="/training-material/stats.html" title="Show view statistics about this repository">
            <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> GTN statistics
        </a>

        <a class="dropdown-item" href="https://plausible.galaxyproject.eu/training.galaxyproject.org?period=12mo&page=/training-material/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html" title="Show view statistics of this page">
            <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> Page Metrics
        </a>

        
            <a class="dropdown-item" href="/training-material/feedback.html" title="Show feedback statistics about this repository">
                <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> GTN feedback
            </a>
        

        <div class="dropdown-item">
            <div>
                <i class="fas fa-palette" aria-hidden="true"></i><span class="visually-hidden">gtn-theme</span> Theme
            </div>

            <div id="theme-selector" data-toggle="buttons">
                <label data-value="default" class="btn btn-secondary">
                    <input type="radio" name="options" id="default" autocomplete="off"> Default
                </label>
                <label data-value="night" class="btn btn-secondary">
                    <input type="radio" name="options" id="night" autocomplete="off"> Night
                </label>
                <label data-value="midnight" class="btn btn-secondary">
                    <input type="radio" name="options" id="midnight" autocomplete="off"> Midnight
                </label>
                <label data-value="rainbow" class="btn btn-secondary">
                    <input type="radio" name="options" id="rainbow" autocomplete="off"> Rainbow
                </label>
                <label data-value="progress" class="btn btn-secondary">
                    <input type="radio" name="options" id="progress" autocomplete="off">üè≥Ô∏è‚Äçüåà
                </label>
                <label data-value="halloween" class="btn btn-secondary">
                    <input type="radio" name="options" id="halloween" autocomplete="off"> üéÉ
                </label>
                <label data-value="straya" class="btn btn-secondary">
                    <input type="radio" name="options" id="downunder" autocomplete="off"> üá¶üá∫
                </label>
            </div>

        </div>

        <div class="dropdown-item">
            <div>
                <i class="fas fa-history" aria-hidden="true"></i><span class="visually-hidden">galaxy-rulebuilder-history</span> Previous Versions
            </div>

            
            <div id="archive-selector">
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-03-01/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html" title="Version 2022-03-01">2022-03-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-02-01/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html" title="Version 2022-02-01">2022-02-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-01-01/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html" title="Version 2022-01-01">2022-01-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/" title="Older Versions">Older Versions</a>
            </div>

        </div>

    </div>
</li>


                    <!-- Search bar-->
                    <li class="nav-item">
                      <div id="navbarSupportedContent" role="search">
                        <!-- Search form -->
                        <form class="form-inline mr-auto" method="GET" action="/training-material/search">
                          <i class="fas fa-search nav-link" aria-hidden="true"></i>
                          <div class="md-form mb-2">
                            <input name="query" class="form-control nicer" type="text" placeholder="Search Tutorials" aria-label="Search">
                          </div>
                        </form>
                      </div>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
</header>

        <div class="container main-content" role="main">
        











<!-- Gitter -->




<script>
  ((window.gitter = {}).chat = {}).options = {
  room: 'Galaxy-Training-Network/Lobby'
  };
</script>
<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>

<script type="application/ld+json">
    


{
  "@context": "http://schema.org",
  "@type": "Course",
  "accessMode": [
    "textual",
    "visual"
  ],
  "accessModeSufficient": [
    "textual",
    "visual"
  ],
  "accessibilityControl": [
    "fullKeyboardControl",
    "fullMouseControl"
  ],
  "accessibilityFeature": [
    "alternativeText",
    "tableOfContents"
  ],
  "accessibilitySummary": "Short descriptions are present but long descriptions will be needed for non-visual users",
  "audience": {
    "@type": "EducationalAudience",
    "educationalRole": "students"
  },
  "citation": {
    "@type": "CreativeWork",
    "name": "Community-Driven Data Analysis Training for Biology",
    "url": "https://doi.org/10.1016/j.cels.2018.05.012"
  },
  "copyrightHolder": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "dateModified": "1970-01-01 00:00:06 +0000",
  "discussionUrl": "https://gitter.im/Galaxy-Training-Network/Lobby",
  "headline": "Statistical analysis of DIA data",
  "interactivityType": "mixed",
  "isAccessibleForFree": true,
  "isFamilyFriendly": true,
  "license": "https://spdx.org/licenses/CC-BY-4.0.html",
  "producer": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "provider": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "sourceOrganization": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "isPartOf": {
    "@type": "CreativeWork",
    "name": "Proteomics",
    "description": "Training material for proteomics workflows in Galaxy",
    "url": "https://training.galaxyproject.org//training-material/topics/proteomics/"
  },
  "courseCode": "proteomics / DIA_Analysis_MSstats / hands-on",
  "learningResourceType": "hands-on tutorial",
  "name": "Hands-on for 'Statistical analysis of DIA data' tutorial",
  "url": "https://training.galaxyproject.org//training-material/topics/proteomics/tutorials/DIA_Analysis_MSstats/tutorial.html",
  "timeRequired": "PT1H",
  "keywords": "DIA",
  "description": "The questions this  addresses are:\n - How to perform statistical analysis on DIA mass spectrometry data?\n - How to detect and quantify differentially abundant proteins in a HEK-Ecoli Benchmark DIA datatset?\n\n\\nThe objectives are:\n - Statistical analysis of a HEK-Ecoli Benchmark DIA dataset\n - Understanding statistical approaches in proteomic analysis\n - Using MSstats to find significantly dysregulated proteins in a Spike-in dataset\n\n",
  "inLanguage": {
    "@type": "Language",
    "name": "English",
    "alternateName": "en"
  },
  "coursePrerequisites": [
    {
      "@type": "CreativeWork",
      "url": "https://training.galaxyproject.org//training-material/topics/introduction/",
      "name": "Introduction to Galaxy Analyses",
      "description": "Introduction to Galaxy Analyses",
      "provider": {
        "@type": "Organization",
        "email": "galaxytrainingnetwork@gmail.com",
        "name": "Galaxy Training Network",
        "url": "https://galaxyproject.org/teach/gtn/"
      }
    },
    {
      "@type": "Course",
      "url": "https://training.galaxyproject.org//training-material/topics/proteomics/tutorials/DIA_lib_OSW/tutorial.html",
      "name": "Library Generation for DIA Analysis",
      "description": "Hands-on for 'Library Generation for DIA Analysis' tutorial",
      "learningResourceType": "hands-on tutorial",
      "interactivityType": "expositive",
      "provider": {
        "@type": "Organization",
        "email": "galaxytrainingnetwork@gmail.com",
        "name": "Galaxy Training Network",
        "url": "https://galaxyproject.org/teach/gtn/"
      }
    },
    {
      "@type": "Course",
      "url": "https://training.galaxyproject.org//training-material/topics/proteomics/tutorials/DIA_Analysis_OSW/tutorial.html",
      "name": "DIA Analysis using OpenSwathWorkflow",
      "description": "Hands-on for 'DIA Analysis using OpenSwathWorkflow' tutorial",
      "learningResourceType": "hands-on tutorial",
      "interactivityType": "expositive",
      "provider": {
        "@type": "Organization",
        "email": "galaxytrainingnetwork@gmail.com",
        "name": "Galaxy Training Network",
        "url": "https://galaxyproject.org/teach/gtn/"
      }
    }
  ],
  "hasPart": [

  ],
  "author": [
    {
      "@type": "Person",
      "name": "Matthias Fahrner"
    },
    {
      "@type": "Person",
      "name": "Melanie F√∂ll"
    }
  ],
  "contributor": [
    {
      "@type": "Person",
      "name": "Matthias Fahrner"
    },
    {
      "@type": "Person",
      "name": "Melanie F√∂ll"
    }
  ],
  "about": [
    {
      "@type": "CreativeWork",
      "name": "Proteomics",
      "description": "Training material for proteomics workflows in Galaxy",
      "url": "https://training.galaxyproject.org//training-material/topics/proteomics/"
    },
    {
      "@type": "DefinedTerm",
      "@id": "http://edamontology.org/topic_0121",
      "inDefinedTermSet": "http://edamontology.org",
      "termCode": "topic_0121",
      "url": "https://bioportal.bioontology.org/ontologies/EDAM/?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Ftopic_0121"
    }
  ],
  "skillLevel": "Intermediate"
}
</script>

<section class="tutorial topic-proteomics">
    <h1 data-toc-skip>Statistical analysis of DIA data</h1>
    


    <div class="contributors-line">Authors: 
<a href="/training-material/hall-of-fame/matthias313/" class="contributor-badge contributor-matthias313"><img src="https://avatars.githubusercontent.com/matthias313?s=27" alt="Avatar">Matthias Fahrner</a>


<a href="/training-material/hall-of-fame/foellmelanie/" class="contributor-badge contributor-foellmelanie"><img src="https://avatars.githubusercontent.com/foellmelanie?s=27" alt="Avatar">Melanie F√∂ll</a>


<a href="https://training.galaxyproject.org/                         contributing/" class="contributor-badge contributor-newcontributors"><i class="fas fa-users" aria-hidden="true"></i><span class="visually-hidden">hall-of-fame</span>Add Contributions!</a>

</div>


    <blockquote class="overview">
        <h3>Overview</h3>
        
        <img alt="Creative Commons License" class="float-right" style="border-width:0; display: inline-block; margin:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" aria-hidden="true" />
        
        <strong><i class="far fa-question-circle" aria-hidden="true"></i> Questions:</strong>
        <ul>
        
        <li><p>How to perform statistical analysis on DIA mass spectrometry data?</p>
</li>
        
        <li><p>How to detect and quantify differentially abundant proteins in a HEK-Ecoli Benchmark DIA datatset?</p>
</li>
        
        </ul>

        <strong><i class="fas fa-bullseye" aria-hidden="true"></i> Objectives: </strong>
        <ul>
        
        <li><p>Statistical analysis of a HEK-Ecoli Benchmark DIA dataset</p>
</li>
        
        <li><p>Understanding statistical approaches in proteomic analysis</p>
</li>
        
        <li><p>Using MSstats to find significantly dysregulated proteins in a Spike-in dataset</p>
</li>
        
        </ul>

        
        <strong><i class="fas fa-check-circle" aria-hidden="true"></i> Requirements:</strong>
        <ul>
        
    <li>
    
        
        
        <a href="/training-material/topics/introduction">Introduction to Galaxy Analyses</a>
        
    
    </li>

        
    <li>
    
        
        
        <a href="/training-material/topics/proteomics">Proteomics</a>
        
            <<strong>Histogramì!¢
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                                       200li> Library Generation for DIA Analysis:
                                                                                                                                 a href="/training-material/topics/proteomics/tutorials/DIA_lib_OSW/tutorial.html"><i class="fas fa-laptopnd filóã>metaQuantome:expandó⁄âmetaQuantome: expanutorial</span> hands-on</a>
                                                                                        200 fewer proteins wer          
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                    
                        
                                             
                                             
                                             
                                             
                                     
                                             
                                             
                                             
                                             
                                             
                                                       200li> DIA Analysis using OpenSwathWorkflow:
                                                                                                                                 a href="/training-material/topics/proteomics/tutorials/DIA_Analysis_OSW/tutorial.html"><i class="fas fa-laptopnd filóã>metaQuantome:expandó⁄âmetaQuantome: expanutorial</span> hands-on</a>
                                                                                        200 fewer proteins wer          
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                    
                        
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                             
                                     
                          
    
    </li>

         (output of th

        
        <div><strong><i class="fas fa-hourglass-half" aria-hidden="true"></i> Time estimation:</strong> 1 hour</div>
        

        
        <div><strong><i class="fas fa-graduation-cap" aria-hidden="true"></i> Level: </strong>
        


 Intermediate <span class="visually-hidden">Intermediate</span>
<span class="level intermediate" title="Intermediate Tutorial">
  <i class="fas fa-graduation-cap" aria-hidden="true"></i> <i class="fas fa-graduation-cap" aria-hidden="true"></i> <i class="fas fa-graduation-cap" aria-hidden="true"></i>
</span>

</div>
        

        
        

        
        <div id="supporting-materials"><strong><i class="fa fa-external-link" aria-hidden="true"></i> Supporting Materials:</strong></div>
        <ul class="supporting_material">
            
                <li class="btn btn-default supporting_material"><a href="/training-material/topics/proteomics/slides/introduction.html" title="Topic Overview slides">
                    <i class="fab fa-slideshare" aria-hidden="true"></i> Topic Overview slides that almonst 1200 a></li>
            

            
                <li class="btn btn-default supporting_material">


<a class="topic-icon" title="Zenodo datasets used in this tutorial" href="https://zenodo.org/record/4307758">
    <i class="far fa-copy" aria-hidden="true"></i> Datasets
</a>

</li>
            

            
                <li class="btn btn-default supporting_material">


    <a class="topic-icon" href="/training-material/topics/proteomics/tutorials/DIA_Analysis_MSstats/workflows/" title="Workflows" alt="Statistical analysis of DIA data workflows">
        <i class="fas fa-share-alt" aria-hidden="true"></i> Workflows
    </a>

</li>
            

            

            

            
            
            
                 <li class="btn btn-default supporting_material">
    <a class="topic-icon" href="/training-material/topics/proteomics/tutorials/DIA_Analysis_MSstats/faqs/" title="Frequently Asked Questions" alt="Tutorial FAQs">
        <i class="far fa-question-circle" aria-hidden="true"></i> FAQs
    </a>
  </li>


            

            <!-- Check the GTN Video Library for recordings of this tutorial or associated slides -->
            











  



            
                <li class="btn btn-default supporting_material">


    <a href="#" class="btn btn-default dropdown-toggle topic-icon" data-toggle="dropdown" aria-expanded="false" title="Where to run the tutorial">
        <i class="fas fa-globe" aria-hidden="true"></i><span class="visually-hidden">instances</span> Available on these Galaxies 
    </a>
    <ul class="dropdown-menu">
    
        <li>
            <a class="dropdown-item" href="https://github.com/galaxyproject/training-material/tree/main/topics/proteomics/docker" title="Docker image for this tutorial">
                <i class="fab fa-docker" aria-hidden="true"></i><span class="visually-hidden">docker_image</span> Docker image
            </a>
        </li>
    
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://africa.usegalaxy.eu/" title="Galaxy Africa">
            Galaxy Africa
        </a>
        
    
        
        <a class="dropdown-item" href="https://india.usegalaxy.eu/" title="Galaxy India">
            Galaxy India
        </a>
        
    
        
    
        
        <a class="dropdown-item" href="https://test.galaxyproject.org/" title="Galaxy Test">
            Galaxy Test
        </a>
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://galaxy.genouest.org" title="Galaxy@GenOuest">
            Galaxy@GenOuest
        </a>
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://streetscience.usegalaxy.eu/" title="Street Science">
            Street Science
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.be/" title="UseGalaxy.be">
            UseGalaxy.be
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.eu" title="UseGalaxy.eu">
            UseGalaxy.eu
        </a>
        
    
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.no/" title="UseGalaxy.no">
            UseGalaxy.no
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.org" title="UseGalaxy.org (Main)">
            UseGalaxy.org (Main)
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.org.au" title="UseGalaxy.org.au">
            UseGalaxy.org.au
        </a>
        
    
        
    
        
    
        
    
        
    
    </ul>


</li>
            
        </ul>
        

        <div><strong><i class="far fa-calendar" aria-hidden="true"></i> Last modification:</strong> Aug 6, 2021 </div>
        <div><strong><i class="fas fa-balance-scale" aria-hidden="true"></i> License:</strong>
            
            <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Tutorial Content is licensed under Creative Commons Attribution 4.0 International License</a>
            
            <a rel="license" href="https://github.com/galaxyproject/training-material/blob/main/LICENSE.md">The GTN Framework is licensed under MIT</a>
        </div>
    </blockquote>

    <div class="container">
        <div class="row">
            <!-- sidebar, which will move to the top on a small screen -->
            <div class="col-sm-2">
                <nav id="toc" data-toggle="toc" class="sticky-top" aria-label="Table of Contents"></nav>
            </div>
            <div class="col-sm-10">
                 

                <h1 class="no_toc" id="introduction">Introduction</h1>

<p>This training covers the statistical analysis of data independent acquisition (DIA) mass spectrometry (MS) data, after successfull identification and quantification of peptides and proteins. We therefore recommend to first go through the  a href="/training-material/topics/proteomics/tutorials/DIA_lib_OSW/tutorial.html">DIA library generation tutorial</a> as well as the  a href="/training-material/topics/proteomics/tutorials/DIA_Analysis_OSW/tutorial.html">DIA analysis tutorial</a>, which teach the principles and characteristics of DIA data analysis.</p>

<p>Modern mass spectrometry approaches enables the identification and quantification of thousands of proteins and tens of thousands of peptides in single measurements. This provides immense potential to in-depth explorative analysis of a variety of biological samples. However, often the number of available samples is limited leading to large proteomic datasets with only a few numbers of replicates or samples per condition. Thus, the statistical analysis remains challenging in such in-depth proteomic studies.</p>

<p>Here we will use <strong>MSstats</strong>, which enables the statistical analysis and processing of proteomic data (<span class="citation"><a href="#Choi2014">Choi <i>et al.</i> 2014</a></span>).</p>

<blockquote class="agenda">
  <h3 id="agenda">Agenda</h3>

  <p>In this tutorial, we will cover:</p>

<ol id="markdown-toc">
  <li><a href="#get-data" id="markdown-toc-get-data">Get data</a></li>
  <li><a href="#statistical-analysis-with-msstats" id="markdown-toc-statistical-analysis-with-msstats">Statistical analysis with <strong>MSstats</strong></a>    <ol>
      <li><a href="#detailed-investigation-of-ecoli-identifications-and-quantifications" id="markdown-toc-detailed-investigation-of-ecoli-identifications-and-quantifications">Detailed investigation of Ecoli identifications and quantifications</a></li>
    </ol>
  </li>
  <li><a href="#statistical-analysis-with-msstats-1" id="markdown-toc-statistical-analysis-with-msstats-1">Statistical analysis with <strong>MSstats</strong></a>    <ol>
      <li><a href="#detailed-investigation-of-ecoli-identifications-and-quantifications-1" id="markdown-toc-detailed-investigation-of-ecoli-identifications-and-quantifications-1">Detailed investigation of Ecoli identifications and quantifications</a></li>
    </ol>
  </li>
</ol>

</blockquote>

<h2 id="get-data">Get data</h2>orT4 sampleóÒÀì_Ånotranslate hands_on">
  Jagtapìé>7
Emma Leithìé>7
data-upload"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Data upload</h3>

  <ol>
    <li>uantome_viz/2.0.0-0"><strong>mefor this tutorial and give it a meaningful name</p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-creating-a-new-history"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Creating a new history</h3>  <p>Click the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> icon at the top of the history panel.</p>  <p>If the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> is missing:</p>  <ol>  <li>Click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> icon (<strong>History options</strong>) on the top of the history panel</li>  <li>Select the option <strong>Create New</strong> from the menu</li></ol></blockquote>
    </li>
    <li>Import the DIA analysis results, the sample annotation and the comparison matrix from <a href="https://zenodo.org/record/4307758">Zenodo</a>
      <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://zenodo.org/record/4307758/files/PyProphet_export.tabular
https://zenodo.org/record/4307758/files/Sample_annot_MSstats.txt
https://zenodo.org/record/4307758/files/Comp_matrix_HEK_Ecoli.txt
https://zenodo.org/record/4307758/files/PyProphet_msstats_input.tabular
</code></pre></div>      </div>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-importing-via-links"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Importing via links</h3>  <ul>  <li>Copy the link location</li>  <li>    <p>Open the <span class="notranslate">Galaxy</span> Upload Manager (<i class="fas fa-upload" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-upload</span> on the top-right of the tool panel)</p>  </li>  <li>Select <strong>Paste/Fetch Data</strong></li>  <li>    <p>Paste the link into the text field</p>  </li>  <li>    <p>Press <strong>Start</strong></p>  </li>  <li><strong>Close</strong> the window</li></ul></blockquote>
    </li>
    <li>
      <p>Once the files are green, rename the sample annotation file in ‚ÄòSample_annot_MSstats‚Äô, the comparison matrix file in ‚ÄòComp_matrix_HEK_Ecoli‚Äô and the two DIA analysis results files in ‚ÄòPyProphet_export‚Äô and ‚ÄòPyProphet_msstats_input‚Äô</p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-lot visualidatasetn finishing this tulightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: yp/metaquandataset</h3>  <ul>  <li>Click on the <i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-pencil</span> <strong>pencil icon</strong> for the dataset to edit its attributes</li>  <li>In the central panel, change the <strong>Name</strong> field</li>  <li>Click the <strong>Save</strong> button</li></ul></blockquote>
    </li>
                         u1 id="statistical-analysis-with-msstats">Statistical analysis with <strong>MSstats</strong></h1ubina Mehtaìé>7Timothy J. Griffinìé>7Pratik Jagtapìé>7
Emma Leithìé>7
performing-statistical-analysis-using-msstats-and-the-tabular-output-from-pyprophet-export"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Performing statistical analysis using <strong>MSstats</strong> and the tabular output from <strong>PyProphet export</strong>metaquantome_viz/metaqdepan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/galaxyp/msstats/msstats/3.20.1.0" title="Tested with toolshed.g2.bx.psu.edu/repos/galaxyp/msstats/msstats/3.20.1.0"><strong>MSstats</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/<span class="notranslate">galaxy</span>p/msstats/msstats/3.20.1.0</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron"http:input sourcircos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawOpenSWATH                  nk 8provided and make -in conditionsìêfil                                                              file</span> <em>‚ÄúOpenSWATH_inputrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawPyProphet_exportw name</li>  <li>Press <kbd>En-in conditionsìêfil                                                              file</span> <em>‚ÄúOpenSWATH_annotationrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawSample_annot_MSstatsw name</li>  <li>Press                                 ="http:Compare Groupsrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawta from th           nk 8provided and make -in conditionsìêfil                                                              file</span> <em>‚ÄúComparison Matrixrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawComp_matrix_HEK_Ecoliw name</li>  <li>Press <aì`,*ó–é≥ D\/How do I loul>

      < metaQuantome has comment">
        <h3 id="comment- creati-data-process-and-group-comparisone comparison of the two Spike-in conditions computed? Are there any differences compared to the selected an: data Process and group comparison</h3>
        <p>During the data process step the peptide intensities are normalized and protein inference is performed. Using a predefined comparison matrix multiple comparisons can be performed.</p>
                        </li>
  </ol>

</blockquote>

<T4 sampleóÒÀì_Å                                       n finishing this tutorial! You have now seen how you can create Circos plots withinì¡ *Circos is a very flexible tool, but this flexibiHow many lines does the PyPyprophet_export.tabular file have? How many lines does the ProcessedData have and do you notice any differences in their structure or format?</li>
    <li>How many proteins were used for the Group comparison? (see ComparisonResult)</li>
  </obers. This database classifies enzymes and organizes the relation"><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden">solution</span> Solution</h3>

    , J. Schein, I.The PyPyprophet_export.tabular has appr. 230.000 lines whereas the ProcessedData has over 1 mio lines and is in the so called long format. Here every individual transition (single m/z value) is reported per row.</li>
      <li>The ComparisonResult has 5022 lines, meaning over 5000 Proteins were compared between the two different Spike-in conditions.</li>
    </ol>

  </blockquote>
sualisation/tutorT4 sampleóÒÀì_Åtip">
  <h3 id="tip-tip-continue-with-results-from-zenodon finishing this tulightbulb" aria-hidden="true"></i><span class="visually-hidden">tip</span> Tip: Continue with results from Zenodo</h3>

  <p>In case the <strong>MSstats</strong> run is not yet finished, the results can be downloaded from Zenodo to be able to continue the tutorial</p>
  <ol>
    <li>Import the files from                  zenodo.org/record/4307758">Zenodo</a>
      <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://zenodo.org/record/4307758/files/MSstats_ComparisonResult_export_tabular.tsv
</code></pre></div>      </div>
    </li>
  </ol>
</blockquote>

<h2 id="detailed-investigation-of-ecoli-identifications-and-quantifications">Detailed investigation of Ecoli identifications and quantifications</h2ubina Mehtaìé>7Timothy J. Griffinìé>7Pratik Jagtapìé>7
Emma Leithìé>7
investigating-ecoli-proteins-in-the-msstats-comparison-results"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Investigating Ecoli proteins in the MSstats comparison result    óÍfì9Rp>Gene Ondepan class="tool" data-tool="Grep1" title="Tested with Grep1"><strong>Select</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Grep1</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúSelect lines fromrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawMSstats_ComparisonResult_export_tabular</code> (output of <strong>MSstats</strong> <i class="fas fa-e expand filóã>metaQuantome:expandó⁄âmetaQuantome: expandóÒÄcókä)</li>
  protei     "http:the patternrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draw(ECOLI)|(log2FC)from thhe control ge T4 at differenti  <li>depan class="tool" data-tool="Filter1" title="Tested with Filter1"><strong>Filter</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Filter1</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúFilter‚Äù</em>: <code class="language-plaintext highlighter-rouge">Select_Ecoli</code> (output of <strong>Select</strong> <i class="fas fa-e expand filóã>metaQuantome:expandó⁄âmetaQuantome: expandóÒÄcókä)</li>
  protei     "http:With following conditionrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawc7!='NA'w name</li>  <li>Pres     "http:Number of header lines to skiprcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draws visualize</em> Vol                  /metaqdepan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4"><strong>Histogram</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúDatasetrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawFilter_Ecoli</code> (output of the previous <strong>Filter</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">ndóÒÄcókä)</li>
  protei     "http:Numerical column for x axisrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawColumn: 3w name</li>  <li>Pres     "http:Number of breaks (bars)rcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draw25w name</li>  <li>Pres     "http:Plot titlircos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawDistribution of Ecoli Protein log FC valuesw name</li>  <li>Pres     "http:Label for x axisrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawlog2 Fold Changefrom thhe control ge T4 a
      < metaQuantome has comment">
        <h3 id="comment- creati-extracting-ecoli-informatione comparison of the two Spike-in conditions computed? Are there any differences compared to the selected an: Extracting Ecoli information</h3>
        <p>First we only select rows containing specific terms such as ‚ÄúEcoli‚Äù from the complete ComparisonResults file. Afterwards, the table is filtered to containg only proteins with valid statistical information (e.g. p-value). Using the log2 Fold change values from all remaining Ecoli proteins we can observe the distribution of log2FC values from the comparison of the two Spike-in conditions.</p>
                        </li>
  </ol>

</blockquote>

<T4 sampleóÒÀì_Å                                        xonomy database. consists of a list of all currently identified taxa and the relationships between them.</p>
         óÍfì9Rp>Gene OnHow many Ecoli proteins were identified and for how many was the p-value for the comparison of the two Spike-in conditions computed?</li>
    <li>How does the distribution of the log2FC values look like? Which Spike-in contained higher amounts of Ecoli and is it possible to see how much more Ecoli was spiked-in?</li>
  </obers. This database classifies enzymes and organizes the relations1"><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden">solution</span> Solution</h3>

    <ol>
      <li>In total, over 800 Ecoli proteins were identified from which 500 have a p-value for the comparison of the two Spike-in conditions.</li>
      <li>We can see a gaussian distribution of the log2FC values around a positive value of 3. Since we compared Spike_in_2 / Spike_in_1 we can directly see that Spike_in_2 contained higher amounts of Ecoli. Furthermore, since the apex of the distribution is around 3 and we compared log2 intensities, we could estimate that Spike_in_2 contained approx. 8-times more Ecoli than Spike_in_1.</li>
    </ol>

  </blockquote>
sualisation/tutorh1 id="statistical-analysis-with-msstats-1">Statistical analysis with <strong>MSstats</strong></h1ubina Mehtaìé>7Timothy J. Griffinìé>7Pratik Jagtapìé>7
Emma Leithìé>7
performing-statistical-analysis-using-msstats-and-the-msstats_input-from-pyprophet-export"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Performing statistical analysis using <strong>MSstats</strong> and the msstats_input from <strong>PyProphet export</strong>metaquantome_viz/metaqdepan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/galaxyp/msstats/msstats/3.20.1.0" title="Tested with toolshed.g2.bx.psu.edu/repos/galaxyp/msstats/msstats/3.20.1.0"><strong>MSstats</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/<span class="notranslate">galaxy</span>p/msstats/msstats/3.20.1.0</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron"http:input sourcircos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawMStats 10 column format                  nk 8provided and make -in conditionsìêfil                                                              file</span> <em>‚ÄúMSstats 10-column inputrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawPyProphet_msstats_inputw name</li>  <li>Press                                 ="http:Compare Groupsrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawta from thhe control ge T4 a
      < metaQuantome has comment">
        <h3 id="comment- creati-msstats-input-formate comparison of the two Spike-in conditions computed? Are there any differences compared to the selected an: MSstats input format</h3>
        <p>For the statistical analysis using MSstats the input must be in the long format, containing all relevant information in 10 predefined columns. The conversion of the <strong>PyProphet export</strong> output can either be done using <strong>MSstats</strong> (as we did above), or during the <strong>Pyprophet export</strong> step by using another R package called swath2stats (<span class="citation"><a href="#Blattmann2016">Blattmann <i>et al.</i> 2016</a></span>). Prior to the conversion the data can be processed and filtered using the swath2stats functionalities.</p>
                        </li>
  </ol>

</blockquote>

<T4 sampleóÒÀì_Å                                        e-in conditions. Here we identify 200 Ecoli proteins less than before, however, the number of proteins for which a p-value was calculateHow many lines does the <code class="language-plaintext highlighter-rouge">PyProphet_msstats_input.tabular</code> file have? How many lines does the ProcessedData have and do you notice any differences in their structure or format?</li>
    <li>How many proteins were used for the Group comparison? And do you already see a difference to the first <strong>MSstats</strong> step?</li>
  </obers. This database classifies enzymes and organizes the relationsì	
ìpì˛

ó¿#˚3visualisation/faqs/">FAQ page for the Visualisation∑Ï$ûkrzywinski2009circos">Krzywinski, M., J. Schein, I.The <code class="language-plaintext highlighter-rouge">PyProphet_msstats_input.tabular</code> has over 870.000 lines and the ProcessedData has over 1 mio lines. Here both files are in the long format, in which every transition is reported per row.</li>
      <li>Here the ComparisonResult has only 3871 lines, meaning that almonst 1200 fewer proteins were used in the comparison of the two spike-in conditions.</li>
    </ol>

  </blockquote>
sualisation/tutorT4 sampleóÒÀì_Åtip">
  <h3 id="tip-tip-continue-with-results-from-zenodo-1n finishing this tulightbulb" aria-hidden="true"></i><span class="visually-hidden">tip</span> Tip: Continue with results from Zenodo</h3>

  <p>In case the <strong>MSstats</strong> run is not yet finished, the results can be downloaded from Zenodo to be able to continue the tutorial</p>
  <ol>
    <li>Import the files from                  zenodo.org/record/4307758">Zenodo</a>
      <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://zenodo.org/record/4307758/files/MSstats_ComparisonResult_msstats_input.tsv
</code></pre></div>      </div>
    </li>
  </ol>
</blockquote>

<h2 id="detailed-investigation-of-ecoli-identifications-and-quantifications-1">Detailed investigation of Ecoli identifications and quantifications</h2ubina Mehtaìé>7Timothy J. Griffinìé>7Pratik Jagtapìé>7
Emma Leithìé>7
investigating-ecoli-proteins-in-the-msstats-comparison-results-1"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Investigating Ecoli proteins in the MSstats comparison result    óÍfì9Rp>Gene Ondepan class="tool" data-tool="Grep1" title="Tested with Grep1"><strong>Select</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Grep1</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúSelect lines fromrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawMSstats_ComparisonResult_msstats_input</code> (output of the second <strong>MSstats</strong> <i class="fas fa-e expand filóã>metaQuantome:expandó⁄âmetaQuantome: expandóÒÄcókä)</li>
  protei     "http:the patternrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draw(ECOLI)|(log2FC)from thhe control ge T4 at differenti  <li>depan class="tool" data-tool="Filter1" title="Tested with Filter1"><strong>Filter</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Filter1</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúFilter‚Äù</em>: <code class="language-plaintext highlighter-rouge">Select_Ecoli</code> (output of the second <strong>Select</strong> <i class="fas fa-e expand filóã>metaQuantome:expandó⁄âmetaQuantome: expandóÒÄcókä)</li>
  protei     "http:With following conditionrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawc7!='NA'w name</li>  <li>Pres     "http:Number of header lines to skiprcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draws visualize</em> Vol                  /metaqdepan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4"><strong>Histogram</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/histogram/histogram_rpy/1.0.4</span></span> axyp/metaquantome_stat/metaquantome_stat/2.0.0-0"><stron-in conditionsìêfil                                                              file</span> <em>‚ÄúDatasetrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawFilter_Ecoli</code> (output of the previous <strong>Filter</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">ndóÒÄcókä)</li>
  protei     "http:Numerical column for x axisrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawColumn: 3w name</li>  <li>Pres     "http:Number of breaks (bars)rcos/conf/../data/data-1.txt
 debuggroup summary 31.57s draw25w name</li>  <li>Pres     "http:Plot titlircos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawDistribution of Ecoli Protein log FC valuesw name</li>  <li>Pres     "http:Label for x axisrcos/conf/../data/data-1.txt
 debuggroup summary 31.57s drawlog2 Fold Changefrom thhe control ge T4 at differenti</ol>

</blockquote>

<T4 sampleóÒÀì_Å                                       -3e comparison of thequestion-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> p>
         óÍfì9Rp>Gene OnHow many Ecoli proteins were identified and for how many was the p-value for the comparison of the two Spike-in conditions computed? Are there any differences compared to the selected and filtered results from the previous <strong>MSstats</strong> step?</li>
    <li>How does the distribution of the log2FC values look like? Are there any differences compared to the selected and filtered results from the previous <strong>MSstats</strong> step?</li>
  </obers. This database classifies enzymes and organizes the relationships between them.</p>
ï—</ol>

ó¿Äó‡&ó‡*óœìŒó†¢Why is it necessary to download metaQuanome dat, J. Schein, I.In total, over 600 Ecoli proteins were identified from which 500 have a p-value for the comparison of the two Spike-in conditions. Here we identify 200 Ecoli proteins less than before, however, the number of proteins for which a p-value was calculated differs only slightly.</li>
      <li>Generally, the two log2FC distribution look very similar, showing a gaussian distribution of the log2FC values around a positive value of 3. There seems to be a slight difference of the apex of the distribution, in the first <strong>MSstats</strong> analysis it seems to be higher than 3, whereas in the second <strong>MSstats</strong> analysis the apex seems to be lower than 3.</li>
    </ol>

  </blockquote>
sualisation/tutoru1 class="no_toc" id="                              figure id="figure-1"<strong>MSstatsì!¢ÛsstaDIA_MSstats_sum.png" alt="DIA_MSstats_pipe. " loading="THì±˝(ófigcaption><span class="figcaption-prefix">Figure 1:</span> All-in one <span class="notranslate">workflow</span> for DIA analysis in <span class="notranslate">Galaxy</span>. The statistical analysis using MSstats is highlighted in green.</figcaption></figure>

<p>Using <strong>MSstats</strong> we were able to identify and quantify differentially regulated proteins between two Spike-in conditions in a HEK/Ecoli Benchmark DIA datatset. Furthermore, the preprocessing of the proteomic data prior to the statistical analysis can directly impact results. Thus, it might be beneficial to try various ways of intermediate data processing and statistical analysis to increase the sensitivity and specificity of the investigation.</pwnlofigure id="figure-2"<strong>MSstatsì!¢ÛsstaDIA_MSstats_galaxy.png" alt="DIA_MSstats_galaxy. " loading="THì±˝(ófigcaption><span class="figcaption-prefix">Figure 2:</span> <span class="notranslate">Galaxy</span> worfklow containing all necessary steps for the statistical analysis of DIA data using MSstats</figcaption></figure>

                 
                                                                                                                                                                                                                       Statistical analyses are crucial for meaningful conclusion in proteomic studies</p>
</li>
                                                      p>MSstats enables statistical analysis of data independent acquisition mass spectrometry data</p>
</li>
                        
                                                                                                                                                                                                                                                                                 proteomics/faqs/">FAQ page for the Proteomics                                                                                                                                                                                                                                                                                                                     <h1 data-toc-skip>Useful literature</h1>
                <p>Further information, including links to documentation and original publications, regarding the tools, analysis techniques and the interpretation of results described in this tutorial can be found <a href="/training-material/topics/proteomics#references">here</a>.                     

                
                                                                                                       Choi2014">Choi, M., C.-Y. Chang, T. Clough, D. Broudy, T. Killeen <i>et al.</i>, 2014 <b>MSstats: an R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments</b>. Bioinformatics 30: 2524‚Äì2526.                             1093/bioinformatics/btu305">10.1093/bioinformatics/btu305</a> <a href="https://academic.oup.com/bioinformatics/bioinformatics/article/2748156/MSstats:">https://academic.oup.com/bioinformatics/bioinformatics/article/2748156/MSstats:</a></li>
<li id="Blattmann2016">Blattmann, P., M. Heusel, and R. Aebersold, 2016 <b>SWATH2stats: An R/bioconductor package to process and convert quantitative SWATH-MS proteomics data for downstream analysis tools</b>. PLoS ONE 11: 1‚Äì7.                             1371/journal.pone.0153160">10.1371/journal.pone.0153160</a></li></o                   

                

                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Statistical analysis of DIA data (Proteomics                                                                                                                                                                                                                                                                                                                                      Matthias Fahrner, Melanie F√∂ll, 2021 <b>Statistical analysis of DIA data                                                                                                       proteomics/tutorials/DIA_Analysis_MSstats                                                                            proteomics/tutorials/DIA_Analysis_MSstats                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        proteomics-DIA_Analysis_MSstats,
author = "Matthias Fahrner and Melanie F√∂ll",
title = "Statistical analysis of DIA data                                                        8",
day = "06                                                                          proteomics/tutorials/DIA_Analysis_MSstats                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
            </div>
        </div>
    </                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    