<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <title>From peaks to genes</title>
        
            <script async defer data-domain="training.galaxyproject.org" src="https://plausible.galaxyproject.eu/js/plausible.js"></script>

        
        <link rel="stylesheet" href="/training-material/assets/css/bootstrap.min.css?v=3">
        <link rel="stylesheet" href="/training-material/assets/css/bootstrap-toc.min.css">
        <link rel="stylesheet" href="/training-material/assets/css/main.css?v=2">
        <script src="https://kit.fontawesome.com/67b3f98409.js" crossorigin="anonymous"></script>
        <link rel="stylesheet" href="/training-material/assets/css/academicons.css">
        <link rel="stylesheet" href="/training-material/assets/css/syntax_highlighting.css">
        <link rel="shortcut icon" href="/training-material/favicon.ico" type="image/x-icon" />
        <link rel="alternate" type="application/atom+xml" href="/training-material/feed.xml" />

        
        
        
        
        
        <meta name="description" content="Galaxy is a scientific workflow, data integration, and da..." />
        <meta property="og:title" content="Galaxy Training: From peaks to genes" />
        <meta property="og:description" content="Galaxy is a scientific workflow, data integration, and da..." />
        <meta property="og:image" content="/training-material/assets/images/GTNLogo1000.png" />
    </head>
    <body data-spy="scroll" data-target="#toc">
        <header>
    <nav class="navbar navbar-expand-md navbar-dark" aria-label="Site Navigation">
        <div class="container">
            <a class="navbar-brand" href="/training-material/">
                <img src="/training-material/assets/images/GTN-60px.png" height="30" alt="Galaxy Training Network logo">
                Galaxy Training!
            </a>

            <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#top-navbar" aria-controls="top-navbar" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="top-navbar">
                <ul class="navbar-nav">
                    <li class="nav-item">
                        
                        <a class="nav-link" href="/training-material/topics/introduction" title="Go back to list of tutorials">
                            <i class="far fa-folder" aria-hidden="true"></i> Introduction to Galaxy Analyses
                        </a>
                        
                    </li>

                    <li class="nav-item dropdown">
                        <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Languages">
                            <i class="fas fa-language" aria-hidden="true"></i><span class="visually-hidden">language</span> Languages
                        </a>
                        <div class="dropdown-menu dropdown-menu-right">
                            
                              <strong class="dropdown-header">Automatic Translations</strong>
            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=fr&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url">
                            Fran√ßais
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=ja&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url">
                            Êó•Êú¨Ë™û
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=es&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url">
                            Espa√±ol
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=pt&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url">
                            Portugu√™s
                            </a>
                            
            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=ar&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url">
                            ÿßŸÑÿπÿ±ÿ®Ÿäÿ©
                            </a>
                            
                            <a class="dropdown-item" href="https://translate.google.com/translate?hl=jp&sl=en&tl=&u=https%3A%2F%2Ftraining.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html&edit-text=&act=url" title="">
                            And more!
                            </a>
                        </div>
                    </li>

                    <li class="nav-item dropdown">
    <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Help">
        <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">help</span> Help
    </a>
    <div class="dropdown-menu dropdown-menu-right">
        <!-- disable Tess for now
        <form method="get" action="https://tess.elixir-europe.org/materials">
            <input type="text" id="search" name="q" value="" style="margin-left: 0.5em;/*! border-radius: 0px; */">
            <input type="hidden" value="Galaxy Training" name="content_provider">
            <input type="submit" value="Search on TeSS" style="width: 92%;border-radius: 0px;margin: 0.5em;background: #f47d20;border: 0px;padding: 0.25em;" class="">
        </form>
        -->

        <a class="dropdown-item" href="/training-material/faq" title="Check our FAQs">
           <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> FAQs
        </a>
        
        <a class="dropdown-item" href="/training-material/topics/introduction/faqs/" title="Check our FAQs for the Introduction to Galaxy Analyses topic">
           <i class="far fa-question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> Topic FAQs
        </a>
        
        <a class="dropdown-item" href="https://help.galaxyproject.org/" title="Discuss on Galaxy Help">
            <i class="far fa-comments" aria-hidden="true"></i><span class="visually-hidden">feedback</span> Galaxy Help Forum
        </a>
        <a class="dropdown-item" href="https://gitter.im/Galaxy-Training-Network/Lobby" title="Discuss on gitter">
           <i class="fab fa-gitter" aria-hidden="true"></i><span class="visually-hidden">gitter</span> Discuss on Gitter
        </a>
    </div>
</li>


                    <li class="nav-item dropdown">
    <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-expanded="false" title="Extras">
        <i class="far fa-star" aria-hidden="true"></i><span class="visually-hidden">galaxy-star</span> Extras
    </a>
    <div class="dropdown-menu dropdown-menu-right">

        
        <a class="dropdown-item" href="https://github.com/galaxyproject/training-material/edit/main/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md" title="Edit on GitHub">
          <i class="fab fa-github" aria-hidden="true"></i><span class="visually-hidden">github</span> Edit on GitHub
        </a>

        <a class="dropdown-item" href="/training-material/stats.html" title="Show view statistics about this repository">
            <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> GTN statistics
        </a>

        <a class="dropdown-item" href="https://plausible.galaxyproject.eu/training.galaxyproject.org?period=12mo&page=/training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html" title="Show view statistics of this page">
            <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> Page Metrics
        </a>

        
            <a class="dropdown-item" href="/training-material/feedback.html" title="Show feedback statistics about this repository">
                <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">galaxy-barchart</span> GTN feedback
            </a>
        

        <div class="dropdown-item">
            <div>
                <i class="fas fa-palette" aria-hidden="true"></i><span class="visually-hidden">gtn-theme</span> Theme
            </div>

            <div id="theme-selector" data-toggle="buttons">
                <label data-value="default" class="btn btn-secondary">
                    <input type="radio" name="options" id="default" autocomplete="off"> Default
                </label>
                <label data-value="night" class="btn btn-secondary">
                    <input type="radio" name="options" id="night" autocomplete="off"> Night
                </label>
                <label data-value="midnight" class="btn btn-secondary">
                    <input type="radio" name="options" id="midnight" autocomplete="off"> Midnight
                </label>
                <label data-value="rainbow" class="btn btn-secondary">
                    <input type="radio" name="options" id="rainbow" autocomplete="off"> Rainbow
                </label>
                <label data-value="progress" class="btn btn-secondary">
                    <input type="radio" name="options" id="progress" autocomplete="off">üè≥Ô∏è‚Äçüåà
                </label>
                <label data-value="halloween" class="btn btn-secondary">
                    <input type="radio" name="options" id="halloween" autocomplete="off"> üéÉ
                </label>
                <label data-value="straya" class="btn btn-secondary">
                    <input type="radio" name="options" id="downunder" autocomplete="off"> üá¶üá∫
                </label>
            </div>

        </div>

        <div class="dropdown-item">
            <div>
                <i class="fas fa-history" aria-hidden="true"></i><span class="visually-hidden">galaxy-rulebuilder-history</span> Previous Versions
            </div>

            
            <div id="archive-selector">
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-03-01/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html" title="Version 2022-03-01">2022-03-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-02-01/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html" title="Version 2022-02-01">2022-02-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/2022-01-01/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html" title="Version 2022-01-01">2022-01-01</a>
            
                <a class="btn btn-warning" href="https://training.galaxyproject.org/archive/" title="Older Versions">Older Versions</a>
            </div>

        </div>

    </div>
</li>


                    <!-- Search bar-->
                    <li class="nav-item">
                      <div id="navbarSupportedContent" role="search">
                        <!-- Search form -->
                        <form class="form-inline mr-auto" method="GET" action="/training-material/search">
                          <i class="fas fa-search nav-link" aria-hidden="true"></i>
                          <div class="md-form mb-2">
                            <input name="query" class="form-control nicer" type="text" placeholder="Search Tutorials" aria-label="Search">
                          </div>
                        </form>
                      </div>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
</header>

        <div class="container main-content" role="main">
        











<!-- Gitter -->




<script>
  ((window.gitter = {}).chat = {}).options = {
  room: 'Galaxy-Training-Network/Lobby'
  };
</script>
<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>

<script type="application/ld+json">
    


{
  "@context": "http://schema.org",
  "@type": "Course",
  "accessMode": [
    "textual",
    "visual"
  ],
  "accessModeSufficient": [
    "textual",
    "visual"
  ],
  "accessibilityControl": [
    "fullKeyboardControl",
    "fullMouseControl"
  ],
  "accessibilityFeature": [
    "alternativeText",
    "tableOfContents"
  ],
  "accessibilitySummary": "Short descriptions are present but long descriptions will be needed for non-visual users",
  "audience": {
    "@type": "EducationalAudience",
    "educationalRole": "students"
  },
  "citation": {
    "@type": "CreativeWork",
    "name": "Community-Driven Data Analysis Training for Biology",
    "url": "https://doi.org/10.1016/j.cels.2018.05.012"
  },
  "copyrightHolder": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "dateModified": "1970-01-01 00:00:03 +0000",
  "discussionUrl": "https://gitter.im/Galaxy-Training-Network/Lobby",
  "headline": "From peaks to genes",
  "interactivityType": "mixed",
  "isAccessibleForFree": true,
  "isFamilyFriendly": true,
  "license": "https://spdx.org/licenses/CC-BY-4.0.html",
  "producer": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "provider": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "sourceOrganization": {
    "@type": "Organization",
    "email": "galaxytrainingnetwork@gmail.com",
    "name": "Galaxy Training Network",
    "url": "https://galaxyproject.org/teach/gtn/"
  },
  "isPartOf": {
    "@type": "CreativeWork",
    "name": "Introduction to Galaxy Analyses",
    "description": "Galaxy is a scientific workflow, data integration, and data and analysis persistence and publishing platform that aims to make computational biology accessible to research scientists that do not have computer programming experience. ",
    "url": "https://training.galaxyproject.org//training-material/topics/introduction/"
  },
  "courseCode": "introduction / galaxy-intro-peaks2genes / hands-on",
  "learningResourceType": "hands-on tutorial",
  "name": "Hands-on for 'From peaks to genes' tutorial",
  "url": "https://training.galaxyproject.org//training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html",
  "timeRequired": "PT3H",
  "description": "The questions this  addresses are:\n - How to use Galaxy?\n - How to get from peak regions to a list of gene names?\n\n\\nThe objectives are:\n - Familiarize yourself with the basics of Galaxy\n - Learn how to obtain data from external sources\n - Learn how to run tools\n - Learn how histories work\n - Learn how to create a workflow\n - Learn how to share your work\n\n",
  "inLanguage": {
    "@type": "Language",
    "name": "English",
    "alternateName": "en"
  },
  "hasPart": [

  ],
  "author": [
    {
      "@type": "Person",
      "name": "Anne Pajon"
    },
    {
      "@type": "Person",
      "name": "Clemens Blank"
    },
    {
      "@type": "Person",
      "name": "B√©r√©nice Batut"
    },
    {
      "@type": "Person",
      "name": "Bj√∂rn Gr√ºning"
    },
    {
      "@type": "Person",
      "name": "Nicola Soranzo"
    },
    {
      "@type": "Person",
      "name": "Dilmurat Yusuf"
    },
    {
      "@type": "Person",
      "name": "Sarah Peter"
    },
    {
      "@type": "Person",
      "name": "Helena Rasche"
    }
  ],
  "contributor": [
    {
      "@type": "Person",
      "name": "Anne Pajon"
    },
    {
      "@type": "Person",
      "name": "Clemens Blank"
    },
    {
      "@type": "Person",
      "name": "B√©r√©nice Batut"
    },
    {
      "@type": "Person",
      "name": "Bj√∂rn Gr√ºning"
    },
    {
      "@type": "Person",
      "name": "Nicola Soranzo"
    },
    {
      "@type": "Person",
      "name": "Dilmurat Yusuf"
    },
    {
      "@type": "Person",
      "name": "Sarah Peter"
    },
    {
      "@type": "Person",
      "name": "Helena Rasche"
    }
  ],
  "about": [
    {
      "@type": "CreativeWork",
      "name": "Introduction to Galaxy Analyses",
      "description": "Galaxy is a scientific workflow, data integration, and data and analysis persistence and publishing platform that aims to make computational biology accessible to research scientists that do not have computer programming experience. ",
      "url": "https://training.galaxyproject.org//training-material/topics/introduction/"
    }
  ],
  "skillLevel": "Introductory"
}
</script>

<section class="tutorial topic-introduction">
    <h1 data-toc-skip>From peaks to genes</h1>
    


    <div class="contributors-line">Authors: 
<a href="/training-material/hall-of-fame/pajanne/" class="contributor-badge contributor-pajanne"><img src="https://avatars.githubusercontent.com/pajanne?s=27" alt="Avatar">Anne Pajon</a>


<a href="/training-material/hall-of-fame/blankclemens/" class="contributor-badge contributor-blankclemens"><img src="https://avatars.githubusercontent.com/blankclemens?s=27" alt="Avatar">Clemens Blank</a>


<a href="/training-material/hall-of-fame/bebatut/" class="contributor-badge contributor-bebatut"><img src="https://avatars.githubusercontent.com/bebatut?s=27" alt="Avatar">B√©r√©nice Batut</a>


<a href="/training-material/hall-of-fame/bgruening/" class="contributor-badge contributor-bgruening"><img src="https://avatars.githubusercontent.com/bgruening?s=27" alt="Avatar">Bj√∂rn Gr√ºning</a>


<a href="/training-material/hall-of-fame/nsoranzo/" class="contributor-badge contributor-nsoranzo"><img src="https://avatars.githubusercontent.com/nsoranzo?s=27" alt="Avatar">Nicola Soranzo</a>


<a href="/training-material/hall-of-fame/dyusuf/" class="contributor-badge contributor-dyusuf"><img src="https://avatars.githubusercontent.com/dyusuf?s=27" alt="Avatar">Dilmurat Yusuf</a>


<a href="/training-material/hall-of-fame/sarah-peter/" class="contributor-badge contributor-sarah-peter"><img src="https://avatars.githubusercontent.com/sarah-peter?s=27" alt="Avatar">Sarah Peter</a>


<a href="/training-material/hall-of-fame/hexylena/" class="contributor-badge contributor-hexylena"><img src="https://avatars.githubusercontent.com/hexylena?s=27" alt="Avatar">Helena Rasche</a>


<a href="https://training.galaxyproject.org/                         contributing/" class="contributor-badge contributor-newcontributors"><i class="fas fa-users" aria-hidden="true"></i><span class="visually-hidden">hall-of-fame</span>Add Contributions!</a>

</div>


    <blockquote class="overview">
        <h3>Overview</h3>
        
        <img alt="Creative Commons License" class="float-right" style="border-width:0; display: inline-block; margin:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" aria-hidden="true" />
        
        <strong><i class="far fa-question-circle" aria-hidden="true"></i> Questions:</strong>
        <ul>
        
        <li><p>How to use Galaxy?</p>
</li>
        
        <li><p>How to get from peak regions to a list of gene names?</p>
</li>
        
        </ul>

        <strong><i class="fas fa-bullseye" aria-hidden="true"></i> Objectives: </strong>
        <ul>
        
        <li><p>Familiarize yourself with the basics of Galaxy</p>
</li>
        
        <li><p>Learn how to obtain data from external sources</p>
</li>
        
        <li><p>Learn how to run tools</p>
</li>
        
        <li><p>Learn how histories work</p>
</li>
        
        <li><p>Learn how to create a workflow</p>
</li>
        
        <li><p>Learn how to share your work</p>
</li>
        
        </ul>

        

        
        <div><strong><i class="fas fa-hourglass-half" aria-hidden="true"></i> Time estimation:</strong> 3 hours</div>
        

        
        <div><strong><i class="fas fa-graduation-cap" aria-hidden="true"></i> Level: </strong>
        


 Introductory <span class="visually-hidden">Introductory</span>
<span class="level introductory" title="Introductory Tutorial">
  <i class="fas fa-graduation-cap" aria-hidden="true"></i> <i class="fas fa-graduation-cap" aria-hidden="true"></i> <i class="fas fa-graduation-cap" aria-hidden="true"></i>
</span>

</div>
        

        
        

        
        <div id="supporting-materials"><strong><i class="fa fa-external-link" aria-hidden="true"></i> Supporting Materials:</strong></div>
        <ul class="supporting_material">
            
                <li class="btn btn-default supporting_material"><a href="/training-material/topics/introduction/slides/introduction.html" title="Topic Overview slides">
                    <i class="fab fa-slideshare" aria-hidden="true"></i> Topic Overview slidesnk will be able to a></li>
            

            
                <li class="btn btn-default supporting_material">


<a class="topic-icon" title="Zenodo datasets used in this tutorial" href="https://           5281/zenodo.1025586">
    <i class="far fa-copy" aria-hidden="true"></i> Datasets
</a>

</li>
            

            
                <li class="btn btn-default supporting_material">


    <a class="topic-icon" href="/training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/workflows/" title="Workflows" alt="From peaks to genes workflows">
        <i class="fas fa-share-alt" aria-hidden="true"></i> Workflows
    </a>

</li>
            

            

            

            
            
            
                 <li class="btn btn-default supporting_material">
    <a class="topic-icon" href="/training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/faqs/" title="Frequently Asked Questions" alt="Tutorial FAQs">
        <i class="far fa-question-circle" aria-hidden="true"></i> FAQs
    </a>
  </li>


            

            <!-- Check the GTN Video Library for recordings of this tutorial or associated slides -->
            











  



            
                <li class="btn btn-default supporting_material">


    <a href="#" class="btn btn-default dropdown-toggle topic-icon" data-toggle="dropdown" aria-expanded="false" title="Where to run the tutorial">
        <i class="fas fa-globe" aria-hidden="true"></i><span class="visually-hidden">instances</span> Available on these Galaxies 
    </a>
    <ul class="dropdown-menu">
    
        <li>
            <a class="dropdown-item" href="https://github.com/galaxyproject/training-material/tree/main/topics/introduction/docker" title="Docker image for this tutorial">
                <i class="fab fa-docker" aria-hidden="true"></i><span class="visually-hidden">docker_image</span> Docker image
            </a>
        </li>
    
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://clipseq.usegalaxy.eu/" title="CLIP-Explorer">
            CLIP-Explorer
        </a>
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://cheminformatics.usegalaxy.eu/" title="ChemicalToolbox">
            ChemicalToolbox
        </a>
        
    
        
        <a class="dropdown-item" href="https://climate.usegalaxy.eu/" title="Climate Science Workbench">
            Climate Science Workbench
        </a>
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://africa.usegalaxy.eu/" title="Galaxy Africa">
            Galaxy Africa
        </a>
        
    
        
        <a class="dropdown-item" href="https://india.usegalaxy.eu/" title="Galaxy India">
            Galaxy India
        </a>
        
    
        
    
        
        <a class="dropdown-item" href="https://test.galaxyproject.org/" title="Galaxy Test">
            Galaxy Test
        </a>
        
    
        
        <a class="dropdown-item" href="https://ecology.usegalaxy.eu/" title="Galaxy for Ecology">
            Galaxy for Ecology
        </a>
        
    
        
    
        
        <a class="dropdown-item" href="https://galaxy.genouest.org" title="Galaxy@GenOuest">
            Galaxy@GenOuest
        </a>
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://hicexplorer.usegalaxy.eu" title="HiCExplorer">
            HiCExplorer
        </a>
        
    
        
        <a class="dropdown-item" href="https://humancellatlas.usegalaxy.eu/" title="Human Cell Atlas">
            Human Cell Atlas
        </a>
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://ml.usegalaxy.eu/" title="Machine Learning Workbench">
            Machine Learning Workbench
        </a>
        
    
        
    
        
        <a class="dropdown-item" href="https://metabolomics.usegalaxy.eu/" title="Metabolomics">
            Metabolomics
        </a>
        
    
        
        <a class="dropdown-item" href="https://metagenomics.usegalaxy.eu/" title="Metagenomics">
            Metagenomics
        </a>
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://nanopore.usegalaxy.eu/" title="NanoGalaxy">
            NanoGalaxy
        </a>
        
    
        
    
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://plants.usegalaxy.eu/" title="Plant Anaylsis Workbench">
            Plant Anaylsis Workbench
        </a>
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://singlecell.usegalaxy.eu/" title="Single Cell Omics workbench">
            Single Cell Omics workbench
        </a>
        
    
        
        <a class="dropdown-item" href="https://streetscience.usegalaxy.eu/" title="Street Science">
            Street Science
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.be/" title="UseGalaxy.be">
            UseGalaxy.be
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.eu" title="UseGalaxy.eu">
            UseGalaxy.eu
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.fr/" title="UseGalaxy.fr">
            UseGalaxy.fr
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.no/" title="UseGalaxy.no">
            UseGalaxy.no
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.org" title="UseGalaxy.org (Main)">
            UseGalaxy.org (Main)
        </a>
        
    
        
        <a class="dropdown-item" href="https://usegalaxy.org.au" title="UseGalaxy.org.au">
            UseGalaxy.org.au
        </a>
        
    
        
    
        
    
        
        <a class="dropdown-item" href="https://virology.usegalaxy.eu/" title="Virology">
            Virology
        </a>
        
    
        
    
    </ul>


</li>
            
        </ul>
        

        <div><strong><i class="far fa-calendar" aria-hidden="true"></i> Last modification:</strong> Nov 3, 2021 </div>
        <div><strong><i class="fas fa-balance-scale" aria-hidden="true"></i> License:</strong>
            
            <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Tutorial Content is licensed under Creative Commons Attribution 4.0 International License</a>
            
            <a rel="license" href="https://github.com/galaxyproject/training-material/blob/main/LICENSE.md">The GTN Framework is licensed under MIT</a>
        </div>
    </blockquote>

    <div class="container">
        <div class="row">
            <!-- sidebar, which will move to the top on a small screen -->
            <div class="col-sm-2">
                <nav id="toc" data-toggle="toc" class="sticky-top" aria-label="Table of Contents"></nav>
            </div>
            <div class="col-sm-10">
                 

                <h1 class="no_toc" id="introduction">Introduction</h1>

<p>We stumbled upon a paper (<span class="citation"><a href="#Li2012">Li <i>et al.</i> 2012</a></span>) called <em>‚ÄúThe histone acetyltransferase MOF is a key regulator of the embryonic stem cell core transcriptional network‚Äù</em>. The paper contains the analysis of possible target genes of an interesting protein called Mof. The targets were obtained by ChIP-seq in mice and the raw data is available through <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268">GEO</a>.
However, the list of genes is neither in the supplement of the paper, nor part of the GEO submission.
The closest thing we could find is a file in GEO containing a list of the regions where the signal is significantly enriched (so called <em>peaks</em>):</p>

<table>
  <tbody>
    <tr>
      <td>1</td>
      <td>3660676</td>
      <td>3661050</td>
      <td>375</td>
      <td>210</td>
      <td>62.0876250438913</td>
      <td>-2.00329386666667</td>
    </tr>
    <tr>
      <td>1</td>
      <td>3661326</td>
      <td>3661500</td>
      <td>175</td>
      <td>102</td>
      <td>28.2950833625942</td>
      <td>-0.695557142857143</td>
    </tr>
    <tr>
      <td>1</td>
      <td>3661976</td>
      <td>3662325</td>
      <td>350</td>
      <td>275</td>
      <td>48.3062708406486</td>
      <td>-1.29391285714286</td>
    </tr>
    <tr>
      <td>1</td>
      <td>3984926</td>
      <td>3985075</td>
      <td>150</td>
      <td>93</td>
      <td>34.1879823073944</td>
      <td>-0.816992</td>
    </tr>
    <tr>
      <td>1</td>
      <td>4424801</td>
      <td>4424900</td>
      <td>100</td>
      <td>70</td>
      <td>26.8023246007435</td>
      <td>-0.66282</td>
    </tr>
  </tbody>
</table>

<p><strong>Table 1</strong> Subsample of the available file</p>

<p>The goal of this exercise is to <strong>turn this list of genomic regions into a list of possible target genes</strong>.</p>

<!--SNIPPET-->                    comment">  <h3 data-toc-skip="" id="-results-may-varyandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF</span> Results may vary</h3>  <p>Your results may be slightly different from the ones presented in this tutorial due to differing versions of tools, reference data, exte<span class="notranslate">rna</span>l databases, or because of stochastic processes in the algorithms.</p></blockquote>

<Length of the flanagenda">
  <h3 id="agenda">Agenda</h3>

  <p>In this tutorial, we will deal with:</p>

<ol id="markdown-toc">
  <li><a href="#pretreatments" id="markdown-toc-pretreatments">Pretreatments</a>    <ol>
      <li><a href="#data-upload" id="markdown-toc-data-upload">Data upload</a></li>
    </ol>
  </li>
  <li><a href="#part-1-naive-approach" id="markdown-toc-part-1-naive-approach">Part 1: Naive approach</a>    <ol>
      <li><a href="#file-preparation" id="markdown-toc-file-preparation">File preparation</a></li>
      <li><a href="#analysis" id="markdown-toc-analysis">Analysis</a></li>
      <li><a href="#visualization" id="markdown-toc-visualization">Visualization</a></li>
      <li><a href="#extracting-workflow" id="markdown-toc-extracting-workflow">Extracting <span class="notranslate">workflow</span></a></li>
    </ol>
  </li>
  <li><a href="#part-2-more-sophisticated-approach" id="markdown-toc-part-2-more-sophisticated-approach">Part 2: More sophisticated approach</a>    <ol>
      <li><a href="#preparation" id="markdown-toc-preparation">Preparation</a></li>
      <li><a href="#create-peak-summit-file" id="markdown-toc-create-peak-summit-file">Create peak summit file</a></li>
      <li><a href="#get-gene-names" id="markdown-toc-get-gene-names">Get gene names</a></li>
      <li><a href="#repeat-workflow" id="markdown-toc-repeat-workflow">Repeat <span class="notranslate">workflow</span></a></li>
    </ol>
  </li>
  <li><a href="#share-your-work" id="markdown-toc-share-your-work">Share your work</a></li>
</ol>

</blockquote>

<h1 id="pretreatments">Pretreatments</h1tools\n ^- Learn how histories work\n - Learn how to create a workflow\n open-galaxy"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Open <span class="notranslate">Galaxy</span></h3>
  <ol>
    <li>Browse to a <span class="notranslate">Galaxy</span> instance: the one recommended by your instructor or one in the list <strong   <li>Again rename the filGalaxy</span> instance</strong> on the head of this page</li>
    <li>
      <p>Log in or register (top panel)</p>

      <p><img src=".und result?ólogin_register.png" alt="Login or Register on the top panelmn1/1.3.0ó 7ì∂9devteam    </li>
  nsoran</blockquote>

</sThe <span class="notranslate">Galaxy</span> interface consist of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the central panel will show the tools and datasets.</p>

<figure id="figure-1"reate peak summit fiógalaxy_interface.png" alt="Galaxy interface screenshot showing history panel on the right, tools panel on the left, and main panel at the centermn1/1.3.0ó 7ì∂9devfigcaption><span class="figcaption-prefix">Figure 1:</span> The <span class="notranslate">Galaxy</span> interface</figcaption></figur>
 
<p>Let‚Äôs start with a fresh history.</p>

<blockquote class="notranslate hands_on">
  —Ωì:¨3p>This tool returnsreate-history"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Create historyusing-the-scratchbookó-e2spectMake sure you have an empty analysis history.</p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-creating-a-new-history"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Creating a new history</h3>  <p>Click the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> icon at the top of the history panel.</p>  <p>If the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> is missing:</p>  <ol>  <li>Click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> icon (<strong>History options</strong>) on the top of the history panel</li>  <li>Select the option <strong>Create New</strong> from the menu</li></ol></blockquote>
    </li>
    <li>
      <p><strong>Rename your history</strong> to make it easy to recognize</promosome nameó–ûìÄ∂;4bgruetip">
        <h3 id="tip-rename-a-history"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden">tip</span> Rename a history</h3>

        <ul>
          <li>
            <p>Click on the title of the history (by default the title is <code class="language-plaintext highlighter-rouge">Unnamed history</code>)</p>

            leó@óì–Round r../../shared/images/rename_history.png" alt="Renaming historymn1/1.3.0ó 7ì∂9devteam                          jli>Type <code class="language-plaintext highlighter-rouge">  <li>Again rename the filGalaxy</span> Introduction</code> as the name</li>
          <li>Press <kbd>Enter</kbd></li>
        </ul>

                        </li>
  </ol>

</blockquote>

<rìqÏ4Amdata-upload">Data upload</h2>

<blockquote class="notranslate hands_on">
  <h3 id="hands_on-hands-on-data-upload"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Data upload</h3>

  <ol>
    <li>Download the list of peak regions (the file <a href="https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE37268&amp;format=file&amp;file=GSE37268%5Fmof3%2Eout%2Ehpeak%2Etxt%2Egz"><code class="language-plaintext highlighter-rouge">GSE37268_mof3.out.hpeak.txt.gz</code></a>) from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268">GEO</a> to your computer</li>
    <li>
      <p>Click on the upload button in the upper left ot the interface</p>

      <p><img src="../../../galaxy-interface/images/upload_button.png" alt="Upload data iconmn1/1.3.0ó 7ì∂9devteam    </li>
    <li>Press <strong>Choose local file</strong> and search for your file on your computer</li>
    <li>Select <code class="language-plaintext highlighter-rouge">interval</code> as <strong>Type</strong></li>
    <li>Press <strong>Start</strong></li>
    <li>Press <strong>Close</strong></li>
    <li>
      <p>Wait for the upload to finish. <span class="notranslate">Galaxy</span> will automatically unpack the file.</p>
    </li>
    <li>
      <p>After this you will see your first history item in <span class="notranslate">Galaxy</span>‚Äôs right pane. It will go through
the gray (preparing/queued) and yellow (running) states to become green (success):</p>

      <p><img src=".und result?óintro_01.png" alt="History sectionmn1/1.3.0ó 7ì∂9devteam/      <p>Directly uploading files is not the only way to get data into <span class="notranslate">Galaxy</span></p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-importing-via-links"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Importing via links</h3>  <ul>  <li>Copy the link location</li>  <li>    <p>Open the <span class="notranslate">Galaxy</span> Upload Manager (<i class="fas fa-upload" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-upload</span> on the top-right of the tool panel)</p>  </li>  <li>Select <strong>Paste/Fetch Data</strong></li>  <li>    <p>Paste the link into the text field</p>  </li>  <li>    <p>Change <strong>Type (set all):</strong> from ‚ÄúAuto-detect‚Äù to <code class="language-plaintext highlighter-rouge">interval</code></p>  </li>  <li>    <p>Press <strong>Start</strong></p>  </li>  <li><strong>Close</strong> the window</li></ul></blockquote>
mosome nameó–ûìÄ∂;4bgruetip">
        <h3 id="tip-tip-importing-data-to-galaxysome additional changes. This is <em>not</em> a hands-on as installing Circos on yotip</span> Tip: Importing data to <span class="notranslate">Galaxy</span></h3>
        <p>There are <a href="/training-material/topics/galaxy-interface/tutorials/get-data/slides.html">more options</a> for advanced users.</p>
                        </li>
  </ol>

</blockquote>

<Length of the flancomment">
  r distribution ainterval-file-formatandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regu`Interval file format</h3>
  <p><strong>Interval</strong> format is a <span class="notranslate">Galaxy</span> format for representing genomic intervals. It is tab-separated, but has the added requirement that three of the columns must be:</p>
  <ul>
    <li>chromosome ID</li>
    <li>start position (0-based)</li>
    <li>end position (end-exclusive)</li>
  </ul>

  <p>An optional strand column can also be specified, and an initial header row can be used to label the columns, which do not have to be in any special order. Unlike <span class="notranslate">BED</span> format (see below) arbitrary additional columns can also be present.</p>

  <p>You can find more information about formats that can be used in <span class="notranslate">Galaxy</span> at the <a href="https://usegalaxy.org/static/formatHelp.html    <li>Again rename the filGalaxy</span> Data Formats page</a>.</p>
</blockquote>
 Length of the flanking region(s)óA±>2000ì—Ωì:¨3p>This tool returninspect-and-edit-attributes-of-a-filvant</li>
    </uló»%In order to compare the two files, we have to make sure that the chromosome names folloInspect and edit attributes of a file</h3>

  <ol>
    <li>-e2spectClick on the file in the history panel</p>

      <p>Some meta-information (e.g. format, reference database) about the file and the header of the file are then displayed, along with the number of lines in the file (48,647):</p>

      <p><img src=".und result?óinput_expanded_file.png" alt="Expanded file in historymn1/1.3.0ó 7ì∂9devteam    </li>
    <li>
      <p>Click on the <i class="fale.óÅCóØ'
2After this you will see your first history item i<span class="notranslate">galaxy</span>-eye</span> (eye) icon (<strong>View data</strong>) in your dataset in the history</p>

      <p>The content of the file is displayed in the central panel</p>
    </li>
    <li>
      <p>Click on the <i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-pencil</span> (pencil) icon (<strong>Edit attributes</strong>) in your dataset in the history</p>

      <p>A form to edit dataset attributes is displayed in the central panel</p>
    </li>
    <li>Search for <code class="language-plaintext highlighter-rouge">mm9</code> in <strong>Database/Build</strong> attribute and select <code class="language-plaintext highlighter-rouge">Mouse July 2007 (NCBI37/mm9)</code> (the paper tells us the peaks are from <code class="language-plaintext highlighter-rouge">mm9</code>)</li>
    <li>Click on <strong>Save</strong> on the top</li>
    <li>
      <p>Add a tag called <code class="language-plaintext highlighter-rouge">#peaks</code> to the dataset to make it easier to track in the history</p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-adding-a-tagsome additional changes. This is <em>not</em> a hands-on as installing Circos on yo</span> Tip: Adding a tagck that the ‚ÄúDatabase/Build‚dataset</li>  <li>Click on <i class="fas fa-tags" aria-hidden="true"></i><span class="visually-hidden">  <li>Again rename the file to somethingtags</span> <strong>Edit dataset tags</strong></li>  <li>    <p>Add a tag starting with <code class="language-plaintext highlighter-rouge">#</code></p>    <p>Tags starting with <code class="language-plaintext highlighter-rouge">#</code> will be automatically propagated to the outputs of tools using this dataset.</p>  </li>  <li>Check that the tag is appearing below the dataset name</li></ul></blockquote>

      <p>The dataset should now look like below in the history</p>

      <p><img src=".und result?óinput_tagged_file.png" alt="Peaks file. " width="250px" height="300px" loading="lazy" />vteam    </li>
  nsoranz/code> as the nameIn order to find the related genes to these peak regions,
we also need a list of genes in mice, which we can obtain from UCSC.</p>

<blockquote class="notranslate hands_on">
  ow to create a workflow\n data-upload-from-ucsc"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Data upload from UCSCusing-the-scratchbookó-e2spectSearch for <code class="language-plaintext highlighter-rouge">UCSC Main</code> in the tool search bar (top left)</p>

      <p><img src=".und result?ó101_01.png" alt="UCSC Main tool in tools sectionmn1/1.3.0ó 7ì∂9devteam    </li>
    <li>
      <p>Click on <code class="language-plaintext highlighter-rouge">UCSC Main</code> <i class="fas fa-ot in any standard format and just by looking at it, we cannot find out </p>

      <p>You will be taken to the <strong>UCSC table browser</strong>, which looks something like this:</p>

      <p><img src=".und result?óintro_02.png" alt="UCSC table browser interfacemn1/1.3.0ó 7ì∂9devteam    </li>
    <li>Set the following options
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8ócladtxt
 debuggroup summary 23.98s processing track_1 scatter circMammalrts. The available to     ó·ˆ8ógenomtxt
 debuggroup summary 23.98s processing track_1 scatter circMouserts. The available to     ó·ˆ8óassembly‚Äù</em>: <code class="language-plaintext highlighter-rouge">July 2007 (NCBI37/mm9)rts. The available to     ó·ˆ8ógroup‚Äù</em>: <code class="language-plaintext highlighter-rouge">Genes and Gene Predictionsrts. The available to     ó·ˆ8ótrack‚Äù</em>: <code class="language-plaintext highlighter-rouge">RefSeq Genesrts. The available to     ó·ˆ8ótabltxt
 debuggroup summary 23.98s processing track_1 scatter circrefGenerts. The available to     ó·ˆ8óregìp
ó‡óÀk
Actions to Apply‚Äù</em>:ìa
ìpïﬁñ3ìp3lgenomerts. The available to     ó·ˆ8óoutput format‚Äù</em>: <code class="language-plaintext highlighter-rouge">  <li>Again rename the filBED</span> - browser extensible datarts. The available to     ó·ˆ8óSend output to‚Äù</em>: <code class="language-plaintext highlighter-rouge">  <li>Again rename the filGalaxy</span></code> (only)e available  entrysoranzoìoC2*  <li>
      <p>Click on the <strong>get output</strong> button</p>

      <p>You will see the next screen:</p>

      <p><img src=".und result?óintro_03.png" alt="Output settingumn1/1.3.0ó 7ì∂9devteam    </li>
    <li>Make sure that <em>‚ÄúCreate one <span class="notranslate">BED</span> record per‚Äù</em> is set to <code class="language-plaintext highlighter-rouge">Whole Gene</code></li>
    <li>Click on the <strong>Send Query to <span class="notranslate">Galaxy</span></strong> button</li>
    <li>Wait for the upload to finish</li>
    <li>
      <p>Rename our dataset to something more recognizable like <code class="language-plaintext highlighter-rouge">Genes</code></p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip->
    <lió‚*;Convert Genomic Intervalsì“BED</span></code></li>
    <li>Press <strong>Convert datatype</strong></li>
    <li>Check that the ‚ÄúDatabase/Build‚Äù is <codóÿ'/mm9</code> (the database build for mice used in the paper)</li>
    <li>Again rename the file to something more recognizable, e.g.ìqÏ4Peak regionsìA~BED</span></code></li>
  </ol>ó1It‚Äôs time to find the overlapping intervals (finally!). To do that,  to <code class="language-plaintext highlighter-rouge">Genes</code></li>  <li>Click the <strong>Save</strong> button</li></ul></blockquote>
    </li>
    <li>Add a tag called <code class="language-plaintext highlighter-rouge">#genes</code> to the dataset to make it easier to track in the history</li>
                         Length of the flancomment">
  r distribution abed-file-formatandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regu`<span class="notranslate">BED</span> file format</h3>
  <p>The <strong   <li>Again rename the filBED</span> - Browser Extensible Data</strong> format provides a flexible way to encode gene regions. <span class="notranslate">BED</span> lines have three required fields:</p>
  <ul>
    <li>chromosome ID</li>
    <li>start position (0-based)</li>
    <li>end position (end-exclusive)</li>
  </ul>

  <p>There can be up to and nine additional optional fields, but the number of fields per line must be consistent throughout any single set of data.</p>

  <p>You can find more information about it at <a href="https://genome.ucsc.edu/FAQ/FAQformat#format1">UCSC</a> including a description of the optional fields.</p>
</blockquote>
 p>Now we have collected all the data we need to start our analysis.</p>

<h1 id="part-1-naive-approach">Part 1: Naive approach</h1>

<p>We will first use a ‚Äúnaive‚Äù approach to try to identify the genes that the peak regions are associated with. We will identify genes that overlap at least 1bp with the peak regions.</p>

<h2 id="file-preparation">File preparation</h2>

<p>Let‚Äôs have a look at our files to see what we actually have here.</p>

<blockquote class="notranslate hands_on">
  ow to create a workflow\n view-file-content"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: View file contentusing-the-scratchbookó-e2spectClick on the <i class="fale.óÅCóØ'
2After this you will see your first history item i<span class="notranslate">galaxy</span>-eye</span> (eye) icon (<strong>View data</strong>) of the peak file to view the content of it</p>

      <p>It should look like this:</p>

      <p><img src=".und result?óintro_04.png" alt="Contents of the peak filemn1/1.3.0ó 7ì∂9devteam    </li>
    <li>
      <p>View the content of the regions of the genes from UCSC</p>

      <p><img src=".und result?óintro_05.png" alt="Contents of UCSC filemn1/1.3.0ó 7ì∂9devteam    </li>
  nsoranz/code> as the naLength of the flan                                       t configuration toìp¸ìêè	 tool.</pói:dCongratulations on finishing this tutorial! You have now seen how you can create Cp>While the file from UCSC has labels for the columns, the peak file does not. Can you guess what the columns stand for?</p>

  <block7" alt="Avata               ish.ìA~)#will automatical"><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden">solution</span> Solution</h3>

    <p>This peak file is not in any standard format and just by looking at it, we cannot find out what the numbers in the different columns mean. In the paper the authors mention that they used the peak caller <a href="https://www.ncbi.nlm.nih.gov/pubmed/20598134">HPeak</a>.</p>

    <p>By looking at the HPeak manual we can find out that the columns contain the following information:</p>

    <ul>
      <li>chromosome name by number</li>
      <li>start coordinate</li>
      <li>end coordinate</li>
      <li>length</li>
      <li>location within the peak that has the highest hypothetical <span class="notranslate">DNA</span> fragment coverage (summit)</li>
      <li>not relevant</li>
      <li>not relevant</li>
    </ul>

  </blockquote>
</blockquote>

<p>In order to compare the two files, we have to make sure that the chromosome names follow the same format.
As we can see, the peak file lacksx;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplchr</code> before any chromosome number. But what happens with chromosome 20 and 21? Will it be X and Y instead? Let‚Äôs check:</p>

<bln ^- Learn how histories work\n - Learn how to create a workflow\n view-end-of-filvant</li>
    </uló»%In order to compare the two files, we have to make sure that the chromosome names folloView end of file</h3>

  <ol>
    <li>Search for <strong>Select last</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span> tool and run <strong>Select last lines from a dataset (tail)</strong>Y instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óText file‚Äù</em>: our peak file <code class="language-plaintext highlighter-rouge">GSE37268_mof3.out.hpeak.txt.gzrts. The available to     ó·ˆ8óOpera
      "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":Keep last linesrts. The available to     ó·ˆ8óNumber of lines‚Äù</em>: Choose a value, e.g. <code class="language-plaintext highlighter-rouge">100rts. The available  entrysoranzoìoC2*  <li>Click <strong>Execute</strong></li>
    <li>Wait for the job to finish</li>
    <li>
      <p>Inspect the file through the npack the file.óÅCóØ'
2After this you will see your first history item i<span class="notranslate">galaxy</span>-eye</span> (eye) icon (<strong>View data</strong>)</promosome nameó–ûìÄ∂;4bgrue                   <h3 id="question-question  //www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268">GEO</a> to your computeró°'lick on the upload button in the upper      <ol>
          <li>How are the chromosomes named?</li>
          <li>How are the chromosomes X and Y named?</li>
        </ol>

                                          ificantly enri#will automaticall1"><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden">solution</span> Solution</h3>
          <oì–OperaóqeAKThe chromosomes are just given by their number. In the gene file from UCSC, they started with <code class="language-plaintext highlighter-rouge">chr</code>e available tools are The chromosomes X and Y are named 20 and 21</li>
          </ol>
        </                  td<em>‚ÄúReplace withì·</ol>
</blockquote>

</sIn order to convert the chromosome names we have therefore two things to do:</p>

<ol>
  <li>add <code class="language-plaintext highlighter-rouge">chr</code></li>
  <li>change 20 and 21 to X and Y</li>
</oltools\n ^- Learn how histories work\n - Learn how to create a workflow\n adjust-chromosome-names"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Adjust chromosome namesusing-the-scratchbookóldpan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3" title="Tested with toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3"><strong>Replace Text</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3</span></span>  in a specific columnY instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óFile to process‚Äù</em>: our peak file <code class="language-plaintext highlighter-rouge">GSE37268_mof3.out.hpeak.txt.gzrts. The available to     ó·ˆ8óin colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":orkflow</span>ì*n,sha<li>
          <p><em>‚ÄúFind patter    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":[0-9]+</code></p>

          <p>This will look for numerical digits</pp>
ó!∂ìœ
ó∞ëóhÅÂ  <li>
          <p><em>‚ÄúReplace with‚Äù</em>: <code class="language-plaintext highlighter-rouge">chr&amp;</code></p>

          <p><code class="language-plaintext highlighter-rouge">&amp;</code> is a placeholder for the find result of the pattern search</pp>
ó!∂ìœ
ó∞ëóhÅÂ%intrysoranzoìoC2*  <li>
      <p>Rename your output file <code class="language-plaintext highlighter-rouge">chr prefix added</code>.</p>
    </li>
    <li><span class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3" title="Tested with toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3"><strong>Replace Text</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3</span></span>   : Let‚Äôs rerun the tool withmm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óFile to process‚Äù</em>: the output from the last run,x;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplchr prefix addedrts. The available to     ó·ˆ8óin colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":orkflow</span>ì*n,sha     ó·ˆ8óFind patter    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":chr20rts. The available to     ó·ˆ8óReplace with‚Äù</em>: <code class="language-plaintext highlighter-rouge">chrXrts. The available </ul>

      <ameó–ûìÄ∂;4bgruetip">
        <h3 id="tip-tip-rerunning-a-toolsome additional changes. This is <em>not</em> a hands-on as installing Circos on yotip</span> Tip: Rerunning a tool</h3>

        <ul>
          <li>Expand the dataset information</li>
          <li>Press the <i class="fas fa-syncde> (the database build for mice used in the paper)</li>
    <li>Again rename the file to somethingrefresh</span> icon (<strong>Run this job again</strong>)</li>
    hyó¡ü8;rophy</s                  </li>
    <li>
      <p>Rename your output file <code class="language-plaintext highlighter-rouge">chrX fixed</code></p>
    </li>
    <li><span class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3" title="Tested with toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3"><strong>Replace Text</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_column/1.1.3</span></span>   : Rerun this tool to do the same for chromosome Y
      <u>ó˛&~ì√øTì0Æ
≥vó·ˆ8óFile to procesheight:12px;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplchrX fixed</code>, the output from the last run                   ó·ˆ8óin colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":orkflow</span>ì*n,sha     ó·ˆ8óFind patter    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":chr21rts. The available to     ó·ˆ8óReplace with‚Äù</em>: <code class="language-plaintext highlighter-rouge">chrYrts. The available  entrysoranzoìoC2*  <li>
      <p>Inspect the latest file through the npack the file.óÅCóØ'
2After this you will see your first history item i<span class="notranslate">galaxy</span>-eye</span> (eye) icon. Have we been successful?</p>

      <p>We have quite a few files now and need to take care to select the correct ones at each step.</promosome nameó–ûìÄ∂;4bgrue                   <h3 id="question-question  neó¶èM.introduction/tutorials/galaxy-intro-peaks2gene∑êŸintroduction-galaxy-intro-peaks2genes,
author = "Anne Pajon and      <p>How many regions are in our output file? You can click the name of the output to expand it and see the number.</p>

                                            icantly enri#will automaticallomics, but can also be used for other types of dataìÅLóˇ”ì	
ìpì˛

ó¿#˚3visualisation/faqs/">FAQ page           <p>It should be equal to the number of regions in your first file, <code class="language-plaintext highlighter-rouge">GSE37268_mof3.out.hpeak.txt.gz</code>: 48,647
If yours says 100 regions, then you have run it on                                                        Tail</code> file and need to re-run the steps.</p>
                            td<em>‚ÄúReplace withì·  <li>Rename the file to something more recognizable, e.g. <code class="language-plaintext highlighter-rouge">Peak regions</code></li>
  </ol>
</blockquote>

<rìqÏ4Amanalysis">Analysis</h2>

<p>Our goal is to compare the 2 region files (the genes file and the peak file from the publication)
to know which peaks are related to which genes. If you only want to know which peaks are located <strong>inside</strong> genes (within the gene body) you
can skip the next step. Otherwise, it might be reasonable to include the <strong>promoter</strong> region of the genes into the comparison, e.g. because
you want to include transcriptions factors in ChIP-seq experiments. There is no strict definition for promoter region but 2kb <span class="notranslate">upstream</span> of the TSS (start of region) is commonly used. We‚Äôll use the <strong>Get Flanks</strong> tool to get regions 2kb bases <span class="notranslate">upstream</span> of the start of the gene to 10kb bases <span class="notranslate">downstream</span> of the start (12kb in length). To do this we tell the Get Flanks tool we want regions <span class="notranslate">upstream</span> of the start, with an offset of 10kb, that are 12kb in length, as shown in the diagram below.</p>

<p><img src="../../images/intro_get_flanks.png" alt="Get Flankumn1/1.3.0ó 7ì∂9devteam/cbln ^- Learn how histories work\n - Learn how to create a workflow\n add-promoter-region-to-gene-records"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Add promoter region to gene recordsusing-the-scratchbookóldpan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/get_flanks/get_flanks1/1.0.0" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/get_flanks/get_flanks1/1.0.0"><strong>Get Flanks</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/get_flanks/get_flanks1/1.0.0</span></span>  returns flanking region/s for every gene,Y instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óSelect data‚Äù</em>: <code class="language-plaintext highlighter-rouge">Genes</code> file from UCSC                   ó·ˆ8óRegìp
ó‡óÀk
Actions to Apply‚Äù</em>:ìa
ìpïﬁñ3ìp3lAround Startrts. The available to     ó·ˆ8óLocation of the flanking region/height:12px;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purpl  <li>Again rename the filUpstream</span>rts. The available to     ó·ˆ8óOffset‚Äù</em>: <code class="language-plaintext highlighter-rouge">i0000rts. The available to     ó·ˆ8óLength of the flanking region(s)‚Äù</em>: <code class="language-plaintext highlighter-rouge">i2000rts. The available </ul>

      <p>This tool returns flanking regions for every gene</p>
    </li>
    <li>
      <p>Compare the rows of the resulting <span class="notranslate">BED</span> file with the input to find out how the start and end positions changed</promosome nameó–ûìÄ∂;4bgruetip">
        <h3 id="tip-tip-inspecting-several-files-using-the-scratchbooksome additional changes. This is <em>not</em> a hands-on as installing Circos on yotip</span> Tip: Inspecting several files using the scratchbook</h3>

        <ul>
          <li>
            <p>Click <strong>Enable/Disable Scratchbook</strong> on the top panel</p>

            leó@óì–Round result?óintro_scratchbook_enable.png" alt="Enable/Disable Scratchbookmn1/1.3.0ó 7ì∂9devteam                          jp>
Click on the <i class="fale.óÅCóØ'
2After this you will see your first history item i<span class="notranslate">galaxy</span>-eye</span> (eye) icon of the files to inspect</li>
          <li>
            <p>Click on <strong>Show/Hide Scratchbook</strong></p               leó@óì–Round result?óintro_scratchbook_show_hide.png" alt="Show/Hide Scratchbookmn1/1.3.0ó 7ì∂9devteam          o view your hiü8;rophy</s                  </li>
    <li>Rename your dataset to reflect your findings (<code class="language-plaintext highlighter-rouge">Promoter regions</code>)</li>
  </ol>
</blockquote>

</sThe output is regions that start from 2kb <span class="notranslate">upstream</span> of the TSS and include 10kb <span class="notranslate">downstream</span>. For input regions on the positive strand e.g.x;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplchr1 134212701 134230065</code> this gives <code class="language-plaintext highlighter-rouge">chr1 134210701 134222701</code>. For regions on the negative strand e.g.x;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplchr1 8349819 9289958</code> this gives <code class="language-plaintext highlighter-rouge">chr1  9279958 9291958</code>.</p>

<p>You might have noticed that the UCSC file is in <code class="language-plaintext highlighter-rouge">  <li>Again rename the filBED</span></code> format and has a database associated to it. That‚Äôs what we want for our peak file as well. The <strong>Intersect</strong> tool we will use can automatically convert interval files to <span class="notranslate">BED</span> format but we‚Äôll convert our interval file explicitly here to show how this can be achieved with <span class="notranslate">Galaxy</span>.</p>

<blockquote class="notranslate hands_on">
  —Ωì:¨3p>This tool returnshange-format-and-databasvant</li>
    </uló»%In order to compare the two files, we have to make sure that the chromosome names followhange format and databaseusing-the-scratchbookóClick on the <i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-pencil</span> (pencil) icon in the history entry of our peak region file</li>
    <li>Switch to the <strong>Convert</strong> tab</li>
    <li>Select <code class="language-plaintext highlighter-rouge">Convert Genomic Intervals to <span class="notranslate">BED</span></code></li>
    <li>Press <strong>Convert datatype</strong></li>
    <li>Check that the ‚ÄúDatabase/Build‚Äù is <code class="language-plaintext highlighter-rouge">mm9</code> (the database build for mice used in the paper)</li>
    <li>Again rename the file to something more recognizable, e.g. <code class="language-plaintext highlighter-rouge">Peak regions <span class="notranslate">BED</span></code></li>
  </ol>
</blockquote>

</sIt‚Äôs time to find the overlapping intervals (finally!). To do that, we want to extract the genes which overlap/intersect with our peaks.</p>

<blockquote class="notranslate hands_on">
  ow to create a workflow\n find-overlaps"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Find Overlapsusing-the-scratchbookóldpan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/intersect/gops_intersect_1/1.0.0" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/intersect/gops_intersect_1/1.0.0"><strong>Intersect</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/intersect/gops_intersect_1/1.0.0</span></span>  the intervals of two datasets,Y instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óRetur    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":Overlapping Intervalsrts. The available to     ó·ˆ8óof‚Äù</em>: the UCSC file with promoter regions (<code class="language-plaintext highlighter-rouge">Promoter regions</code>)                   ó·ˆ8óthat intersect‚Äù</em>: our peak region file from <strong>Replace</strong> (<code class="language-plaintext highlighter-rouge">Peak regions <span class="notranslate">BED</span></code>)                   ó·ˆ8ófor at least‚Äù</em>: <code class="language-plaintext highlighter-rouge">irkflow</span>ì*n,s</ul>

      <ameó–ûìÄ∂;4bgruecomment">
        <h3 id="comment-cross tsandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regu`lator os</h3>
        <p>The order of the inputs is important! We want to end up with a list of <strong>genes</strong>, so the corresponding dataset with the gene information needs to be the first input (<code class="language-plaintext highlighter-rouge">Promoter regions</code>).</p>
                          <p><img src=".und result?óintro_overlapping_genes.png" alt="Genes overlapping peakumn1/1.3.0ó 7ì∂9devteam    </li>
  nsoran</blockquote>

</sWe now have the list of genes (column 4) overlapping with the peak regions, similar to shown above.</p>

<p>To get a better overview of the genes we obtained, we want to look at their distribution across the different chromosomes.
We will group the table by chromosome and count the number of genes with peaks on each chromosome</p>

<bln ^- Learn how histories work\n - Learn how to create a workflow\n count-genes-on-different-chromosomes"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Count genes on different chromosomesusing-the-scratchbookóldpan class="tool" data-tool="Grouping1" title="Tested with Grouping1"><strong>Group</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Grouping1</span></span>   data by a column and perform aggregate operation on other columnsY instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óSelect data‚Äù</em> to the result of the intersection                   ó·ˆ8óGroup by column‚Äù</em>:"httpì@98introduction/"
    }
  ],
  "skillLevel":Column 1rts. The available to<li>Press <strong>Insert Operation</strong> and chooseata-upload-from-ucscónData upl      Typtxt
 debuggroup summary 23.98s processing track_1 scatter circCountrts. The available tools are ce: theOn colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":Column 1rts. The available tools are ce: theRound result to nearest integer?‚Äù</em>: <code class="language-plaintext highlighter-rouge">nd datasets.ì†y3galaxy_ij√∂rn Gr√ºningìˇB0nsoranzoìul>

      <ameó–ûìÄ∂;4bgrue                   <h3 id="question-questions-3andey, J. S. Byun, question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> utton in the upper      <p>Which chromosome contained the highest number of target genes?</p>

                                            icantly enri#will automatically unpack the file.óÅCóØ'
2After this you will see your first history item inìA~(‚Äôs right pane. It will go t      <p>The result varies with different settings, for example, the annotation may change due to updates at UCSC. If you followed step by step, with the same annotation, it should be chromosome 11 with 1992 genes. Note that for reproducibility, you should keep all input data used because <span class="notranslate">Galaxy</span> can store all parameters but inputs may change e.g. the annotation from UCSC.</p>
                            td<em>‚ÄúReplace withì·</ol>

</blockquote>

<rìqÏ4Amvisualization">Visualization</h2>

<p>Since we have some nice data, let‚Äôs draw a barchart out of it!</p>

<bln ^- Learn how histories work\n - Learn how to create a workflow\n draw-barchart"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Draw barchartusing-the-scratchbookóClick on <i class="fas fa-chart-bar" aria-hidden="true"></i><span class="visually-hidden">  <li>Again rename the file to somethingbarchart</span> (visualize) icon on the output from the <strong>Group</strong> tool</li>
    <li>Select <code class="language-plaintext highlighter-rouge">Bar diagram</code></li>
    <li>Choose a title at <strong>Provide a title</strong>, e.g. <code class="language-plaintext highlighter-rouge">Gene counts per chromosomerts. The availabl<li>Switch to the <i class="fas fa-database" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-chart-select-data</span> <strong>Select data</strong> tab and play around with the settings</li>
    <li>
      <p>When you are happy, click the <i class="far fa-sav                                                          <li>Again rename the file to somethingsave</span> <strong>Save</strong> visualization in the top right of the <em>main panel</em></p>

      <p>This will store it to your saved visualisations where you can later view, download, or share it with others.</p)soranzoìoC2*nsoranz/code> as the narìqÏ4Amextracting-workflow">Extracting <span class="notranslate">workflow</span></h2>

<p>When you look carefully at your history, you can see that it contains all the steps of our analysis, from the beginning to the end. By building this history we have actually built a complete record of our analysis with <span class="notranslate">Galaxy</span> preserving all parameter settings applied at every step.
Wouldn‚Äôt it be nice to just convert this history into a <span class="notranslate">workflow</span> that we‚Äôll be able to execute again and again?</p>

<p   <li>Again rename the filGalaxy</span> makes this very simple with                                                        Extract <span class="notranslate">workflow</span></code> option. This means that any time you want to build a <span class="notranslate">workflow</span>, you can just perform it manually once, and then convert it to a <span class="notranslate">workflow</span>, so that next time it will be a lot less work to do the same analysis. It also allows you to easily share or publish your analysis.</p>

<blockquote class="notranslate hands_on">
  ow to create a workflow\n extract-workflow"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Extract <span class="notranslate">workflow</span>using-the-scratchbookó-e2spect<strong>Clean up</strong> your history: remove any failed (red) jobs from your history by clickinge/Build‚Äù is <codóÿ'/mtimes" aria-hidden="true"></i><span class="visually-hidden">  <li>Again rename the file to somethingcross</span> button.</p>

      <p>This will make the creation of the <span class="notranslate">workflow</span> easier.</p>
    </li>
    <li>
      <p>Click on <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> (<strong>History options</strong>) at the top of your history panel and select <strong>Extract <span class="notranslate">workflow</span></strong>.</p>

      <p><img src=".und result?óhistory_menu_extract_workflow.png" alt="`Extract Workflow` entry in the history options menumn1/1.3.0ó 7ì∂9devteam/      <p>The central panel will show the content of the history in reverse order (oldest on top), and you will be able to choose which steps to include in the <span class="notranslate">workflow</span>.</p>
    </li>
    <li>
      <p>Replace the <strong   <li>Again rename the filWorkflow</span> name</strong> to something more descriptive, for examplepx;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplFrom peaks to genes</code></p>
    </li>
    <li>
      <p>If there are any steps that shouldn‚Äôt be included in the <span class="notranslate">workflow</span>, you can <strong>uncheck</strong> them in the first column of boxes.</p>

      <p>Since we did some steps which where specific to our custom peak file, we might want to exclude:</pp summary 38.92s drawing <strong>Select last</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span></li>
        <   all <strong>Replace Text</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span> steps                   strong>Convert Genomic Intervals to <span class="notranslate">BED</span></strong>                   strong>Get flanks</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span></li>
       entrysoranzoìoC2*  <li>
      <p>Click on the <strong>Create <span class="notranslate">Workflow</span></strong> button near the top.</p>

      <p>You will get a message that the <span class="notranslate">workflow</span> was created. But where did it go?</p>
    </li>
    <li>
      <p>Click on <strong   <li>Again rename the filWorkflow</span></strong> in the top menu of <span class="notranslate">Galaxy</span></p>

      <p>Here you have a list of all your workflows</p>
    </li>
    <li>
      <p>Select the newly generated <span class="notranslate">workflow</span> and click on <strong>Edit</strong></p         <p>You should see something similar to this:</p>

      <p><img src=".und result?óintro_06.png" alt="Editing workflow interfacemn1/1.3.0ó 7ì∂9devteam/osome nameó–ûìÄ∂;4bgruecomment">
        <h3 id="comment-the-workflow-editorandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regu`The <span class="notranslate">workflow</span> editor</h3>
        <p>We can examine the <span class="notranslate">workflow</span> in <span class="notranslate">Galaxy</span>‚Äôs <span class="notranslate">workflow</span> editor. Here you can view/change the parameter settings of each step, add and remove tools, and connect an output from one tool to the input of another, all in an easy and graphical manner. You can also use this editor to build workflows from scratch.</p>
                           <p>Although we have our two inputs in the <span class="notranslate">workflow</span> they are missing their connection to the first tool (<strong>Intersect</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span>), because we didn‚Äôt carry over some of the intermediate steps.</p>
    </li>
    <li>Connect each input dataset to the <strong>Intersect</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span> tool by dragging the arrow pointing outwards on the right of its box (which denotes an output) to an arrow on the left of the <strong>Intersect</strong> box pointing inwards (which denotes an input)</li>
    <li>Rename the input datasets toÀk
Actions to Apply‚Äù</em>:ìa
ìpïﬁñ3ìp3lReference regions</code> and <code class="language-plaintext highlighter-rouge">Peak regions</code></li>
    <li>Click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> (gear) icon at the top right</li>
    <li>Press <strong>Auto Re-layout</strong> to clean up our view
eate peak summit fióintro_07.png" alt="Auto re-layoutingmn1/1.3.0ó 7ì∂9devtli>
    <li>Click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> (gear) icon at the top right</li>
    <li>Press <strong>Save</strong> to save your changes</li>
  </o/strong>ó°'Wait for thtip">
    <h3 id="tip-tip-hiding-intermediate-stepssome additional changes. This is <em>not</em> a hands-on as installing Circos on yotip</span> Tip: Hiding intermediate steps</h3>
    <p>When a <span class="notranslate">workflow</span> is executed, the user is usually primarily interested in the final product and not in all intermediate steps. By default all the outputs of a <span class="notranslate">workflow</span> will be shown, but we can explicitly tell <span class="notranslate">Galaxy</span> which output to show and which to hide for a given <span class="notranslate">workflow</span>. This behaviour is controlled by the little asterisk next to every output dataset:</p>

    <p><img src="../../../../shared/images/workflow_editor_mark_output.png" alt="Workflow editor mark outputmn1/1.3.0ó 7ì∂9devteam/    <p>If you click on this asterisk for any of the output datasets, then <em>only</em> files with an asterisk will be shown, and all outputs without an asterisk will be hidden (Note that clicking <em>all</em> outputs has the same effect as clicking <em>none</em> of the outputs, in both cases all the datasets will be shown).</p>
 Rasche, Crist√≥bal Gallardo, 2021 Now it‚Äôs time to reuse our <span class="notranslate">workflow</span> for a more sophisticated approach.</p>

<h1 id="part-2-more-sophisticated-approach">Part 2: More sophisticated approach</h1>

<p>In part 1 we used an overlap definition of 1 bp (default setting) to identify genes associated with the peak regions. However, the peaks could be broad, so instead, in order to get a more meaningful definition, we could identify the genes that overlap where most of the <span class="notranslate">reads</span> are concentrated, the <strong>peak summit</strong>. We will use the information on the position of the peak summit contained in the original peak file and check for overlap of the summits with genes.</p>

<h2 id="preparation">Preparation</h2>

<p>We again need our peak file, but we‚Äôd like to work in a clean history. Instead of uploading it twice, we can copy it to a new history.</p>

<blockquote class="notranslate hands_on">
  —Ωì:¨3p>This tool returnsopy-history-items"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Copy history itemsusing-the-scratchbookó-e2specting several files usiand give it a new name like <code class="language-plaintext highlighter-rouge">  <li>Again rename the filGalaxy</span> Introduction Part 2</code></p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-creating-a-new-history"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Creating a new history</h3>  <p>Click the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> icon at the top of the history panel.</p>  <p>If the <i class="fas fa-plus" aria-hidden="true"></i><span class="visually-hidden">new-history</span> is missing:</p>  <ol>  <li>Click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> icon (<strong>History options</strong>) on the top of the history panel</li>  <li>Select the option <strong>Create New</strong> from the menu</li></ol></blockquote>
    </li>
    <li>
      <p>Click on the <strong>View all histories</strong> (<i class="fas fa-columns" aria-hidden="true"></i><span class="visually-hidden">  <li>Again rename the file to somethingcolumns</span> icon) at the top right of your history</p>

      <p>You should see both of your histories side-by-side now</p>
    </li>
    <li>Drag and drop the edited peak file (<code class="language-plaintext highlighter-rouge">Peak regions</code>, after the replace steps), which contains the summit information, to your new history.</li>
    <li>Click on the Home icon <i class="fas fa-hom                                                          <li>Again rename the file to somethinghome</span> (or <strong>Analyze Data</strong> on older <span class="notranslate">Galaxy</span> versions) in the top menu bar to go back to your analysis window</li>
                         rìqÏ4Amcreate-peak-summit-file">Create peak summit file</h2>

<p>We need to generate a new <span class="notranslate">BED</span> file from the original peak file that contains the positions of the peak summits. The start of the summit is the start of the peak (column 2) plus the location within the peak that has the highest hypothetical <span class="notranslate">DNA</span> fragment coverage (column 5). As the end of the peak region, we will simply define <code class="language-plaintext highlighter-rouge">start + 1</code>.</p>

<blockquote class="notranslate hands_on">
  —Ωì:¨3p>This tool returnsreate-peak-summit-file"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Create peak summit file</h3>

  <ol>
    <li>ldpan class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0"><strong>Compute an <span class="notranslate">expression <strongon every row</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0</span></span> Y instead? Let‚Äôs check:</pìƒˆIview-end-of-filóÒVoView endAdd <span class="notranslate">expression <stron‚Äù</em>: <code class="language-plaintext highlighter-rouge">c2+c5rts. The available to     ó·ˆ8óas a new column to‚Äù</em>: our peak file <code class="language-plaintext highlighter-rouge">Peak regions</code> (the interval format file)                   ó·ˆ8óRound result?reatments">Pretreatments</h1óë{Iopen-galaxyónOpenìA~( </ESrts. The available oì,D)soranzoìoC2*nsoranz  <p>This will create an 8th column in our table, which we will use in our next step:</pg-the-scratchbookó-e2spectRename the output <code class="language-plaintext highlighter-rouge">Peak regions new column</code></p>
    </li>
    <li><span class="tool" data-tool="toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0" title="Tested with toolshed.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0"><strong>Compute an <span class="notranslate">expression <strongon every row</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: <span class="notranslate">toolshed</span>.g2.bx.psu.edu/repos/devteam/column_maker/Add_a_column1/1.3.0</span></span>  rerun this tool on the last result with
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óAdd <span class="notranslate">expression <stron‚Äù</em>: <code class="language-plaintext highlighter-rouge">c8+1rts. The available to     ó·ˆ8óas a new column to‚Äù</em>:                                                        Peak regions new column</code> file we just created                   ó·ˆ8óRound result?reatments">Pretreatments</h1óë{Iopen-galaxyónOpenìA~( </ESrts. The available  entrysoranzoìoC2*  <li>Rename this file <code class="language-plaintext highlighter-rouge">Peak summit regions</code></li>
  </ol>
</blockquote>

</sNow we cut out just the chromosome plus the start and end of the summit:</p>

<bln ^- Learn how histories work\n - Learn how to create a workflow\n cut-out-columns"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Cut out columns</h3>
  <ol>
    <li><span class="tool" data-tool="Cut1" title="Tested with Cut1"><strong>Cut</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: Cut1</span></span>  columns from a table  instead? Let‚Äôs csettings
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óCut columnheight:12px;margin--right:5px;">¬†¬†¬†¬†¬†</span> (light purplc1,c8,c9rts. The available to     ó·ˆ8óDelimited by Tab‚Äù</em>: <code class="language-plaintext highlighter-rouge">Tabrts. The available to     ó·ˆ8óFrom‚Äù</em>: <code class="language-plaintext highlighter-rouge">Peak summit regionsrts. The available </ul>

      <p>The output from <strong>Cut</strong> will be in <code class="language-plaintext highlighter-rouge">tabular</code> format.</p>
    </li>
    <li>
      <p>Change the format to <code class="language-plaintext highlighter-rouge">interval</code> (use the <i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-pencil</span>) since that‚Äôs what the tool <strong>Intersect</strong> expects.</p>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-changing-the-datatype"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Changing the datatype</h3>  <ul>  <li>Click on the <i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-pencil</span> <strong>pencil icon</strong> for the dataset to edit its attributes</li>  <li>In the central panel, click on the <i class="fas fa-database" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-chart-select-data</span> <strong>Datatypes</strong> tab on the top</li>  <li>Select <code class="language-plaintext highlighter-rouge">interval</code></li>  <li>Click the <strong>Save</strong> button</li></ul></blockquote>

      <p>The output should look like below:</p>

      <p><img src=".und result?óintro_summits.png" alt="Peak summits. " width="200px" loading="lazy" />vteam    </li>
  nsoran</blockquote>

<rìqÏ4Amget-gene-names">Get gene names</h2>

<p>The RefSeq genes we downloaded from UCSC did only contain the RefSeq identifiers, but not the gene names. To get a list of gene names in the end, we use another <span class="notranslate">BED</span> file from the Data Libraries.</p>

<blockquote class="comment">
  <h3 id="comment-comments-1andey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regu`lator os</h3>
  <p>There are several ways to get the gene names in, if you need to do it yourself. One way is to retrieve a <span class="notranslate">mapping</span> through Biomart and then join the two files (<strong>Join two Datasets side by side on a specified field</strong> <i class="fas fa-wrench" aria-hidden="true"></i><span class="visually-hidden">tool</span>). Another is to get the full RefSeq table from UCSC and manually convert it to <span class="notranslate">BED</span> format.</p>
</blockquote>
 Length of the flanking region(s)óA±>2000ì—Ωì:¨3p>This tool returndata-upload-1"><i class="fa                                                                                                Data upload</h3>

  <ol>
    <li>-e2spect<span class="tool" data-tool="upload1" title="Tested with upload1"><strong>Import</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: upload1</span></span>  <code class="language-plaintext highlighter-rouge">mm9.RefSeq_genes_from_UCSC.<span class="notranslate">bed</span></code> from <a href="https://zenodo.org/record/1025586">Zenodo</a> or from the data library:</p>

      <div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>https://zenodo.org/record/1025586/files/mm9.RefSeq_genes_from_UCSC.<span class="notranslate">bed</span>
</code></pre></div>      </div>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-importing-via-links"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Importing via links</h3>  <ul>  <li>Copy the link location</li>  <li>    <p>Open the <span class="notranslate">Galaxy</span> Upload Manager (<i class="fas fa-upload" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-upload</span> on the top-right of the tool panel)</p>  </li>  <li>Select <strong>Paste/Fetch Data</strong></li>  <li>    <p>Paste the link into the text field</p>  </li>  <li>    <p>Change <strong>Genome</strong> to <code class="language-plaintext highlighter-rouge">mm9</code></p>  </li>  <li>    <p>Press <strong>Start</strong></p>  </li>  <li><strong>Close</strong> the window</li></ul></blockquote>

      <!--SNIPPET-->
      <blockquote class="tip">  <h3 data-toc-skip="" id="-tip-importing-data-from-a-data-library"><i class="far fa-lightbulb" aria-hidden="true"></i><span class="visually-hidden"></span> Tip: Importing data from a data library</h3>  <p>As an alte<span class="notranslate">rna</span>tive to uploading the data from a URL or your computer, the files may also have been made available from a <em>shared data library</em>:</p>  <ul>  <li>Go into <strong>Shared data</strong> (top panel) then <strong>Data libraries</strong></li>  <li>    <p>Navigate to</p>    <p><em>Click on ‚ÄúTraining data‚Äù and then ‚ÄúIntroduction - From peaks to genes‚Äù</em> or  the correct folder as indicated by your instructor</p>  </li>  <li>Select the desired files</li>  <li>Click on the <strong>To History</strong> button near the top and select <strong>as Datasets</strong> from the dropdown menu</li>  <li>In the pop-up window, select the history you want to import the files to (or create a new one)</li>  <li>Click on <strong>Import</strong></li></ul></blockquote>

      <p>As default, <span class="notranslate">Galaxy</span> takes the link as name, so rename them.</p>
    </li>
    <li>
      <p>Inspect the file content to check if it contains gene names.
It should look similar to below:
eate peak summit fióintro_gene_names.png" alt="Gene nameumn1/1.3.0ó 7ì∂9devteam    </li>
    <li>Rename it <code class="language-plaintext highlighter-rouge">mm9.RefSeq_genes</code></li>
    <li>Apply the tag <code class="language-plaintext highlighter-rouge">#genes</code>  Introduction</code> as the narìqÏ4Amrepeat-workflow">Repeat <span class="notranslate">workflow</span></h2>

<p>It‚Äôs time to reuse the <span class="notranslate">workflow</span> we created earlier.</p>

<blockquote class="notranslate hands_on">
  ow to create a workflow\n run-a-workflow"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Run a <span class="notranslate">workflow</span></h3>
  <ol>
    <li>Open the <span class="notranslate">workflow</span> menu (top menu bar)</li>
    <li>Find the <span class="notranslate">workflow</span> you made in the previous section, and select the option <strong>Run</strong></li>
    <li>Choose as inputs our <code class="language-plaintext highlighter-rouge">mm9.RefSeq_genes</code> (<code class="language-plaintext highlighter-rouge">#genes</code>) <span class="notranslate">BED</span> file and the result of the <strong>Cut</strong> tool (<code class="language-plaintext highlighter-rouge">#peaks</code>)</li>
    <li>
      <p>Click <strong>Run <span class="notranslate">workflow</span></strong></p         <p>The outputs should appear in the history but it might take some time until they are finished.</p)soranzoìoC2*nsoranz/code> as the nameWe used our <span class="notranslate">workflow</span> to rerun our analysis with the peak summits. The <strong>Group</strong> tool again produced a list containing the number of genes found in each chromosome.
But wouldn‚Äôt it be more interesting to know the number of peaks in each unique gene? Let‚Äôs rerun the <span class="notranslate">workflow</span> with different settings!</p>

<bln ^- Learn how histories work\n - Learn how to create a workflow\n run-a-workflow-with-changed-settings"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Run a <span class="notranslate">workflow</span> with changed settings</h3>
  <ol>
    <li>Open the <span class="notranslate">workflow</span> menu (top menu bar)</li>
    <li>Find the <span class="notranslate">workflow</span> you made in the previous section, and select the option <strong>Run</strong></li>
    <li>Choose as inputs our <code class="language-plaintext highlighter-rouge">mm9.RefSeq_genes</code> (<code class="language-plaintext highlighter-rouge">#genes</code>) <span class="notranslate">BED</span> file and the result of the <strong>Cut</strong> tool (<code class="language-plaintext highlighter-rouge">#peaks</code>)</li>
    <li>Click on the title of the <i class="fas fa-ot in any standard format and just by looking at it, we cannot find out w<strong>Group</strong> tool to expand the options.</li>
    <li>Change the following settings by clickinge/Build‚Äù is <codóÿ'/mm9</code> (the database build for mice used in the paper)</li>
    <li>Again rename the file to something more recogniz(pencil) icon on the left
mm9</code>ó˛&~ì√øTì0Æ
≥vó·ˆ8óGroup by colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":7rts. The available to    from UCSC.Operationó¿{data-upload-from-ucscónData upload frOn colum    "url": "httpì@98introduction/"
    }
  ],
  "skillLevel":7rts. The available toolj√∂rn Gr√ºningìˇB0nsoranzoìntrysoranzoìoC2*  <li>Click <strong>Run <span class="notranslate">workflow</span></strong></li>
  </ol>
</blockquote>

</sCongratulations! You should have a file with all the unique gene names and a count on how many peaks they contained.</p>

<blockquote class="                                       -4andey, J. S. Byun, question-circle" aria-hidden="true"></i><span class="visually-hidden">question</span> utton in the upper p>The list of unique genes is not sorted. Try to sort it on your own!</p>

  <block7" alt="Avata               ish.ìA~)#will automaticallmic Intervalsì“BED</span></strong>óAóó›¶
Get flanksóëÅoì=ÒóØ'lick on the <strong>Creató,WWork    <p>You can use the tool ‚ÄúSort data in ascending or descending order‚Äù on column 2 and ‚Äúfast numeric sort‚Äù.</p>
  </blockquote>
</blockquote>

<r1 id="share-your-work">Share your work</h1>

<p>One of the most important features of <span class="notranslate">Galaxy</span> comes at the end of an analysis. When you have published striking findings, it is important that other researchers are able to reproduce your in-silico experiment. <span class="notranslate">Galaxy</span> enables users to easily share their workflows and histories with others.</p>

<p>To share a history, click on the <i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> (gear) symbol in the history pane and select <code class="language-plaintext highlighter-rouge">Share or Publish</code>. On this page you can do 3 things:</p>

<ol>
  <li>
    <p><strong>Make accessible via Link</strong></p        p>This generates a link that you can give out to others. Anybody with this link will be able to view your history.</p>
  </li>
  <li>
    <p><strong>Publish History</strong></p        p>This will not only create a link, but will also publish your history. This means your history will be listed under <code class="language-plaintext highlighter-rouge">Shared Data ‚Üí Published Histories</code> in the top menu.</p>
  </li>
  <li>
    <p><strong>Share with Individual Users</strong></p        p>This will share the history only with specific users on the <span class="notranslate">Galaxy</span> instance.</p>
  </li>
</oltools\n ^- Learn how histories work\n - Learn how to create a workflow\n share-history-and-workflow"><i class="fas fa-pencil-alt" aria-hidden="true"></i><span class="visually-hidden">hands_on</span> Hands-on: Share history and <span class="notranslate">workflow</span>using-the-scratchbookóShare one of your histories with your neighbour</li>
    <li>See if you can do the same with your <span class="notranslate">workflow</span>!</li>
    <li>
      <p>Find the history and/or <span class="notranslate">workflow</span> shared by your neighbour</p>

      <p>Histories shared with specific users can be accessed by those users in their history menu (<i class="fas fa-cog" aria-hidden="true"></i><span class="visually-hidden"><span class="notranslate">galaxy</span>-gear</span> (gear) icon) under <code class="language-plaintext highlighter-rouge">Histories shared with me</code>.</p)soranzoìoC2*nsoranz/code> as the nah1 class="no_toc" id="                                <i class="fas fa-trophyany standard format and just by looking at it, we cannotrophy</span> You have just performed your first analysis in <span class="notranslate">Galaxy</span>. You also created a <span class="notranslate">workflow</span> from your analysis so you can easily repeat the exact same analysis on other datasets. Additionally you shared your results and methods with others.                                                                                                                                                                                                                                               Galaxy provides an easy-to-use graphical user interface for often complex commandline tools</p>
</                                                            Galaxy keeps a full record of your analysis in a history</p>
</                                                            Workflows enable you to repeat your analysis on different data</p>
</                                                            Galaxy can connect to external sources for data import and visualization purposes</p>
</                                                            Galaxy provides ways to share your results and methods with others</p>
</                            
                    </                                                                                                                                                                                                                                                           introduction/faqs/">FAQ page for the Introduction to Galaxy Analyses                                                                                                                                                                                                                                                                                                                                                                                                                              Li2012">Li, X., L. Li, R. Pandey, J. S. Byun, K. Gardner <i>et al.</i>, 2012 <b>The Histone Acetyltransferase MOF Is a Key Regulator of the Embryonic Stem Cell Core Transcriptional Network</b>. Cell Stem Cell 11: 163‚Äì178.                                 /j.stem.2012.04.023">10.1016/j.stem.2012.04.023</a></li></ol>
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 From peaks to genes (Introduction to Galaxy Analyses                                                                                                                                                                                                                                                                                                                                      Anne Pajon, Clemens Blank, B√©r√©nice Batut, Bj√∂rn Gr√ºning, Nicola Soranzo, Dilmurat Yusuf, Sarah Peter, Helena Rasche, 2021 <b>From peaks to gene                                                                                                        introduction/tutorials/galaxy-intro-peaks2gene                                                                             introduction/tutorials/galaxy-intro-peaks2gene                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         introduction-galaxy-intro-peaks2genes,
author = "Anne Pajon and Clemens Blank and B√©r√©nice Batut and Bj√∂rn Gr√ºning and Nico               Dilmurat Yusuf and Sarah Peter and Helena Rasche",
title = "From peaks to gene                                                        11",
day = "03                                                                          introduction/tutorials/galaxy-intro-peaks2gene                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      